.storybook/docs-root.css

```diff
@@ -1,24 +1,26 @@
-#docs-root .sbdocs-content {
+#storybook-docs .sbdocs-content {
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the FluentUIV0 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32018&commitId=45f7f5017a992110bd8536ef0864c7b3e84b380b&env=prod&clientType=FLUENTUI)


  <details><summary><b>Accordion</b> 19 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Accordion.AccordionDefaultBsize.chromium.png|0| Added|
|Accordion.AccordionDefaultExample.chromium.png|0| Added|
|Accordion.AccordionExample.chromium.png|0| Added|
|Accordion.AccordionExampleCustomTitle.chromium.png|0| Added|
|Accordion.AccordionExampleRtl.chromium.png|0| Added|
|Accordion.AccordionExclusiveExample.chromium.png|0| Added|
|Accordion.AccordionExclusiveExpandedExample.chromium.png|0| Added|
|Accordion.AccordionPanelCustomContentExample.chromium.png|0| Added|
|Accordion.AccordionPanelCustomTitleExample.chromium.png|0| Added|
|Accordion.Accordion Default Bsize.chromium.png|0| Removed|
|Accordion.Accordion Default Example.chromium.png|0| Removed|
|Accordion.Accordion Example Custom Title.Focuses the accordion title.chromium.png|0| Removed|
|Accordion.Accordion Example Custom Title.chromium.png|0| Removed|
|Accordion.Accordion Example Rtl.chromium.png|0| Removed|
|Accordion.Accordion Example.chromium.png|0| Removed|
|Accordion.Accordion Exclusive Example.chromium.png|0| Removed|
|Accordion.Accordion Exclusive Expanded Example.chromium.png|0| Removed|
|Accordion.Accordion Panel Custom Content Example.chromium.png|0| Removed|
|Accordion.Accordion Panel Custom Title Example.chromium.png|0| Removed|

</details>


  <details><summary><b>Alert</b> 94 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Alert.AlertDefaultBsize.chromium.png|0| Added|
|Alert.AlertExample.chromium.png|0| Added|
|Alert.AlertExampleChildrenRtl.chromium.png|0| Added|
|Alert.AlertExampleDanger.chromium.png|0| Added|
|Alert.AlertExampleDangerTeams.chromium.png|0| Added|
|Alert.AlertExampleDangerTeamsDark.chromium.png|0| Added|
|Alert.AlertExampleDangerTeamsHighContrast.chromium.png|0| Added|
|Alert.AlertExampleDismissAction.chromium.png|0| Added|
|Alert.AlertExampleDismissActionRtl.chromium.png|0| Added|
|Alert.AlertExampleDismissActions.chromium.png|0| Added|
|Alert.AlertExampleDismissible.chromium.png|0| Added|
|Alert.AlertExampleDismissibleTeams.chromium.png|0| Added|
|Alert.AlertExampleDismissibleTeamsDark.chromium.png|0| Added|
|Alert.AlertExampleDismissibleTeamsHighContrast.chromium.png|0| Added|
|Alert.AlertExampleFitted.chromium.png|0| Added|
|Alert.AlertExampleHeader.chromium.png|0| Added|
|Alert.AlertExampleIcon.chromium.png|0| Added|
|Alert.AlertExampleImportantMessage.chromium.png|0| Added|
|Alert.AlertExampleInfo.chromium.png|0| Added|
|Alert.AlertExampleInfoTeams.chromium.png|0| Added|
|Alert.AlertExampleInfoTeamsDark.chromium.png|0| Added|
|Alert.AlertExampleInfoTeamsHighContrast.chromium.png|0| Added|
|Alert.AlertExampleOof.chromium.png|0| Added|
|Alert.AlertExampleOofTeams.chromium.png|0| Added|
|Alert.AlertExampleOofTeamsDark.chromium.png|0| Added|
|Alert.AlertExampleOofTeamsHighContrast.chromium.png|0| Added|
|Alert.AlertExampleRtl.chromium.png|0| Added|
|Alert.AlertExampleShorthand.chromium.png|0| Added|
|Alert.AlertExampleSuccess.chromium.png|0| Added|
|Alert.AlertExampleUrgent.chromium.png|0| Added|
|Alert.AlertExampleUrgentTeams.chromium.png|0| Added|
|Alert.AlertExampleUrgentTeamsDark.chromium.png|0| Added|
|Alert.AlertExampleUrgentTeamsHighContrast.chromium.png|0| Added|
|Alert.AlertExampleWarning.chromium.png|0| Added|
|Alert.AlertExampleWidth.chromium.png|0| Added|
|Alert.Alert Default Bsize.chromium.png|0| Removed|
|Alert.Alert Example Children Rtl.chromium.png|0| Removed|
|Alert.Alert Example Danger Teams Dark.chromium.png|0| Removed|
|Alert.Alert Example Danger Teams High Contrast.chromium.png|0| Removed|
|Alert.Alert Example Danger Teams.chromium.png|0| Removed|
|Alert.Alert Example Danger.chromium.png|0| Removed|
|Alert.Alert Example Dismiss Action Rtl.chromium.png|0| Removed|
|Alert.Alert Example Dismiss Action.chromium.png|0| Removed|
|Alert.Alert Example Dismiss Actions.chromium.png|0| Removed|
|Alert.Alert Example Dismissible Teams Dark.chromium.png|0| Removed|
|Alert.Alert Example Dismissible Teams High Contrast.chromium.png|0| Removed|
|Alert.Alert Example Dismissible Teams.chromium.png|0| Removed|
|Alert.Alert Example Dismissible.chromium.png|0| Removed|
|Alert.Alert Example Fitted.chromium.png|0| Removed|
|Alert.Alert Example Header.chromium.png|0| Removed|
|Alert.Alert Example Icon.chromium.png|0| Removed|
|Alert.Alert Example Important Message.chromium.png|0| Removed|
|Alert.Alert Example Info Teams Dark.chromium.png|0| Removed|
|Alert.Alert Example Info Teams High Contrast.chromium.png|0| Removed|
|Alert.Alert Example Info Teams.chromium.png|0| Removed|
|Alert.Alert Example Info.chromium.png|0| Removed|
|Alert.Alert Example Oof Teams Dark.Focuses the action button.chromium.png|0| Removed|
|Alert.Alert Example Oof Teams Dark.Hovers the action button.chromium.png|0| Removed|
|Alert.Alert Example Oof Teams Dark.Hovers the focused action button.chromium.png|0| Removed|
|Alert.Alert Example Oof Teams Dark.chromium.png|0| Removed|
|Alert.Alert Example Oof Teams High Contrast.Focuses the action button.chromium.png|0| Removed|
|Alert.Alert Example Oof Teams High Contrast.Hovers the action button.chromium.png|0| Removed|
|Alert.Alert Example Oof Teams High Contrast.Hovers the focused action button.chromium.png|0| Removed|
|Alert.Alert Example Oof Teams High Contrast.chromium.png|0| Removed|
|Alert.Alert Example Oof Teams.Focuses the action button.chromium.png|0| Removed|
|Alert.Alert Example Oof Teams.Hovers the action button.chromium.png|0| Removed|
|Alert.Alert Example Oof Teams.Hovers the focused action button.chromium.png|0| Removed|
|Alert.Alert Example Oof Teams.chromium.png|0| Removed|
|Alert.Alert Example Oof.Focuses the action button.chromium.png|0| Removed|
|Alert.Alert Example Oof.Hovers the action button.chromium.png|0| Removed|
|Alert.Alert Example Oof.Hovers the focused action button.chromium.png|0| Removed|
|Alert.Alert Example Oof.chromium.png|0| Removed|
|Alert.Alert Example Rtl.chromium.png|0| Removed|
|Alert.Alert Example Shorthand.chromium.png|0| Removed|
|Alert.Alert Example Success.chromium.png|0| Removed|
|Alert.Alert Example Urgent Teams Dark.Focuses the action button.chromium.png|0| Removed|
|Alert.Alert Example Urgent Teams Dark.Hovers the action button.chromium.png|0| Removed|
|Alert.Alert Example Urgent Teams Dark.Hovers the focused action button.chromium.png|0| Removed|
|Alert.Alert Example Urgent Teams Dark.chromium.png|0| Removed|
|Alert.Alert Example Urgent Teams High Contrast.Focuses the action button.chromium.png|0| Removed|
|Alert.Alert Example Urgent Teams High Contrast.Hovers the action button.chromium.png|0| Removed|
|Alert.Alert Example Urgent Teams High Contrast.Hovers the focused action button.chromium.png|0| Removed|
|Alert.Alert Example Urgent Teams High Contrast.chromium.png|0| Removed|
|Alert.Alert Example Urgent Teams.Focuses the action button.chromium.png|0| Removed|
|Alert.Alert Example Urgent Teams.Hovers the action button.chromium.png|0| Removed|
|Alert.Alert Example Urgent Teams.Hovers the focused action button.chromium.png|0| Removed|
|Alert.Alert Example Urgent Teams.chromium.png|0| Removed|
|Alert.Alert Example Urgent.Focuses the action button.chromium.png|0| Removed|
|Alert.Alert Example Urgent.Hovers the action button.chromium.png|0| Removed|
|Alert.Alert Example Urgent.Hovers the focused action button.chromium.png|0| Removed|
|Alert.Alert Example Urgent.chromium.png|0| Removed|
|Alert.Alert Example Warning.chromium.png|0| Removed|
|Alert.Alert Example Width.chromium.png|0| Removed|
|Alert.Alert Example.chromium.png|0| Removed|

</details>


  <details><summary><b>Animation</b> 16 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Animation.AnimationExample.chromium.png|0| Added|
|Animation.AnimationExampleDelay.chromium.png|0| Added|
|Animation.AnimationExampleDirection.chromium.png|0| Added|
|Animation.AnimationExampleDuration.chromium.png|0| Added|
|Animation.AnimationExampleFillMode.chromium.png|0| Added|
|Animation.AnimationExampleIterationCount.chromium.png|0| Added|
|Animation.AnimationExampleTimingFunction.chromium.png|0| Added|
|Animation.AnimationExampleVisible.chromium.png|0| Added|
|Animation.Animation Example Delay.chromium.png|0| Removed|
|Animation.Animation Example Direction.chromium.png|0| Removed|
|Animation.Animation Example Duration.chromium.png|0| Removed|
|Animation.Animation Example Fill Mode.chromium.png|0| Removed|
|Animation.Animation Example Iteration Count.chromium.png|0| Removed|
|Animation.Animation Example Timing Function.chromium.png|0| Removed|
|Animation.Animation Example Visible.chromium.png|0| Removed|
|Animation.Animation Example.chromium.png|0| Removed|

</details>


  <details><summary><b>Attachment</b> 62 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Attachment.AttachmentActionExampleShorthand.chromium.png|0| Added|
|Attachment.AttachmentActionExampleShorthandTeams.chromium.png|0| Added|
|Attachment.AttachmentActionExampleShorthandTeamsDark.chromium.png|0| Added|
|Attachment.AttachmentActionExampleShorthandTeamsHighContrast.chromium.png|0| Added|
|Attachment.AttachmentActionableExampleShorthand.chromium.png|0| Added|
|Attachment.AttachmentActionableExampleShorthandTeams.chromium.png|0| Added|
|Attachment.AttachmentActionableExampleShorthandTeamsDark.chromium.png|0| Added|
|Attachment.AttachmentActionableExampleShorthandTeamsHighContrast.chromium.png|0| Added|
|Attachment.AttachmentDefaultBsize.chromium.png|0| Added|
|Attachment.AttachmentDescriptionExampleShorthand.chromium.png|0| Added|
|Attachment.AttachmentExampleRtl.chromium.png|0| Added|
|Attachment.AttachmentExampleShorthand.chromium.png|0| Added|
|Attachment.AttachmentHeaderExampleShorthand.chromium.png|0| Added|
|Attachment.AttachmentIconExampleShorthand.chromium.png|0| Added|
|Attachment.AttachmentProgressExampleShorthand.chromium.png|0| Added|
|Attachment.Attachment Action Example Shorthand Teams Dark.Focuses action.chromium.png|0| Removed|
|Attachment.Attachment Action Example Shorthand Teams Dark.Focuses root.chromium.png|0| Removed|
|Attachment.Attachment Action Example Shorthand Teams Dark.Hovers action.chromium.png|0| Removed|
|Attachment.Attachment Action Example Shorthand Teams Dark.Hovers root.chromium.png|0| Removed|
|Attachment.Attachment Action Example Shorthand Teams Dark.chromium.png|0| Removed|
|Attachment.Attachment Action Example Shorthand Teams High Contrast.Focuses action.chromium.png|0| Removed|
|Attachment.Attachment Action Example Shorthand Teams High Contrast.Focuses root.chromium.png|0| Removed|
|Attachment.Attachment Action Example Shorthand Teams High Contrast.Hovers action.chromium.png|0| Removed|
|Attachment.Attachment Action Example Shorthand Teams High Contrast.Hovers root.chromium.png|0| Removed|
|Attachment.Attachment Action Example Shorthand Teams High Contrast.chromium.png|0| Removed|
|Attachment.Attachment Action Example Shorthand Teams.Focuses action.chromium.png|0| Removed|
|Attachment.Attachment Action Example Shorthand Teams.Focuses root.chromium.png|0| Removed|
|Attachment.Attachment Action Example Shorthand Teams.Hovers action.chromium.png|0| Removed|
|Attachment.Attachment Action Example Shorthand Teams.Hovers root.chromium.png|0| Removed|
|Attachment.Attachment Action Example Shorthand Teams.chromium.png|0| Removed|
|Attachment.Attachment Action Example Shorthand.Focuses action.chromium.png|0| Removed|
|Attachment.Attachment Action Example Shorthand.Focuses root.chromium.png|0| Removed|
|Attachment.Attachment Action Example Shorthand.Hovers action.chromium.png|0| Removed|
|Attachment.Attachment Action Example Shorthand.Hovers root.chromium.png|0| Removed|
|Attachment.Attachment Action Example Shorthand.chromium.png|0| Removed|
|Attachment.Attachment Actionable Example Shorthand Teams Dark.Focuses action.chromium.png|0| Removed|
|Attachment.Attachment Actionable Example Shorthand Teams Dark.Focuses root.chromium.png|0| Removed|
|Attachment.Attachment Actionable Example Shorthand Teams Dark.Hovers action.chromium.png|0| Removed|
|Attachment.Attachment Actionable Example Shorthand Teams Dark.Hovers root.chromium.png|0| Removed|
|Attachment.Attachment Actionable Example Shorthand Teams Dark.chromium.png|0| Removed|
|Attachment.Attachment Actionable Example Shorthand Teams High Contrast.Focuses action.chromium.png|0| Removed|
|Attachment.Attachment Actionable Example Shorthand Teams High Contrast.Focuses root.chromium.png|0| Removed|
|Attachment.Attachment Actionable Example Shorthand Teams High Contrast.Hovers action.chromium.png|0| Removed|
|Attachment.Attachment Actionable Example Shorthand Teams High Contrast.Hovers root.chromium.png|0| Removed|
|Attachment.Attachment Actionable Example Shorthand Teams High Contrast.chromium.png|0| Removed|
|Attachment.Attachment Actionable Example Shorthand Teams.Focuses action.chromium.png|0| Removed|
|Attachment.Attachment Actionable Example Shorthand Teams.Focuses root.chromium.png|0| Removed|
|Attachment.Attachment Actionable Example Shorthand Teams.Hovers action.chromium.png|0| Removed|
|Attachment.Attachment Actionable Example Shorthand Teams.Hovers root.chromium.png|0| Removed|
|Attachment.Attachment Actionable Example Shorthand Teams.chromium.png|0| Removed|
|Attachment.Attachment Actionable Example Shorthand.Focuses action.chromium.png|0| Removed|
|Attachment.Attachment Actionable Example Shorthand.Focuses root.chromium.png|0| Removed|
|Attachment.Attachment Actionable Example Shorthand.Hovers action.chromium.png|0| Removed|
|Attachment.Attachment Actionable Example Shorthand.Hovers root.chromium.png|0| Removed|
|Attachment.Attachment Actionable Example Shorthand.chromium.png|0| Removed|
|Attachment.Attachment Default Bsize.chromium.png|0| Removed|
|Attachment.Attachment Description Example Shorthand.chromium.png|0| Removed|
|Attachment.Attachment Example Rtl.chromium.png|0| Removed|
|Attachment.Attachment Example Shorthand.chromium.png|0| Removed|
|Attachment.Attachment Header Example Shorthand.chromium.png|0| Removed|
|Attachment.Attachment Icon Example Shorthand.chromium.png|0| Removed|
|Attachment.Attachment Progress Example Shorthand.chromium.png|0| Removed|

</details>


  <details><summary><b>Avatar</b> 30 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar.AvatarDefaultBsize.chromium.png|0| Added|
|Avatar.AvatarExampleExcludedInitialsShorthand.chromium.png|0| Added|
|Avatar.AvatarExampleGetInitialsShorthand.chromium.png|0| Added|
|Avatar.AvatarExampleIconShorthand.chromium.png|0| Added|
|Avatar.AvatarExampleImageShorthand.chromium.png|0| Added|
|Avatar.AvatarExampleLabelShorthand.chromium.png|0| Added|
|Avatar.AvatarExampleNameShorthand.chromium.png|0| Added|
|Avatar.AvatarExampleRtl.chromium.png|0| Added|
|Avatar.AvatarExampleShorthand.chromium.png|0| Added|
|Avatar.AvatarExampleSizeShorthand.chromium.png|0| Added|
|Avatar.AvatarExampleSquare.chromium.png|0| Added|
|Avatar.AvatarExampleStatusCustomizationShorthand.chromium.png|0| Added|
|Avatar.AvatarExampleStatusImageShorthand.chromium.png|0| Added|
|Avatar.AvatarExampleStatusShorthand.chromium.png|0| Added|
|Avatar.AvatarUsageExampleShorthand.chromium.png|0| Added|
|Avatar.Avatar Default Bsize.chromium.png|0| Removed|
|Avatar.Avatar Example Excluded Initials Shorthand.chromium.png|0| Removed|
|Avatar.Avatar Example Get Initials Shorthand.chromium.png|0| Removed|
|Avatar.Avatar Example Icon Shorthand.chromium.png|0| Removed|
|Avatar.Avatar Example Image Shorthand.chromium.png|0| Removed|
|Avatar.Avatar Example Label Shorthand.chromium.png|0| Removed|
|Avatar.Avatar Example Name Shorthand.chromium.png|0| Removed|
|Avatar.Avatar Example Rtl.chromium.png|0| Removed|
|Avatar.Avatar Example Shorthand.chromium.png|0| Removed|
|Avatar.Avatar Example Size Shorthand.chromium.png|0| Removed|
|Avatar.Avatar Example Square.chromium.png|0| Removed|
|Avatar.Avatar Example Status Customization Shorthand.chromium.png|0| Removed|
|Avatar.Avatar Example Status Image Shorthand.chromium.png|0| Removed|
|Avatar.Avatar Example Status Shorthand.chromium.png|0| Removed|
|Avatar.Avatar Usage Example Shorthand.chromium.png|0| Removed|

</details>


  <details><summary><b>Box</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Box.BoxDefaultBsize.chromium.png|0| Added|
|Box.BoxExample.chromium.png|0| Added|
|Box.BoxShorthandExample.chromium.png|0| Added|
|Box.Box Default Bsize.chromium.png|0| Removed|
|Box.Box Example.chromium.png|0| Removed|
|Box.Box Shorthand Example.chromium.png|0| Removed|

</details>


  <details><summary><b>Breadcrumb</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Breadcrumb.BreadcrumbExample.chromium.png|0| Added|
|Breadcrumb.BreadcrumbExampleIconDivider.chromium.png|0| Added|
|Breadcrumb.BreadcrumbExampleSizes.chromium.png|0| Added|
|Breadcrumb.Breadcrumb Example Icon Divider.chromium.png|0| Removed|
|Breadcrumb.Breadcrumb Example Sizes.chromium.png|0| Removed|
|Breadcrumb.Breadcrumb Example.chromium.png|0| Removed|

</details>


  <details><summary><b>Button</b> 168 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button.ButtonDefaultBsize.chromium.png|0| Added|
|Button.ButtonExample.chromium.png|0| Added|
|Button.ButtonExampleCircular.chromium.png|0| Added|
|Button.ButtonExampleContentAndIcon.chromium.png|0| Added|
|Button.ButtonExampleContentAndIconTeams.chromium.png|0| Added|
|Button.ButtonExampleContentAndIconTeamsDark.chromium.png|0| Added|
|Button.ButtonExampleContentAndIconTeamsHighContrast.chromium.png|0| Added|
|Button.ButtonExampleDisabled.chromium.png|0| Added|
|Button.ButtonExampleDisabledFocusable.chromium.png|0| Added|
|Button.ButtonExampleDisabledTeams.chromium.png|0| Added|
|Button.ButtonExampleDisabledTeamsDark.chromium.png|0| Added|
|Button.ButtonExampleDisabledTeamsHighContrast.chromium.png|0| Added|
|Button.ButtonExampleEmphasis.chromium.png|0| Added|
|Button.ButtonExampleFlat.chromium.png|0| Added|
|Button.ButtonExampleFluid.chromium.png|0| Added|
|Button.ButtonExampleIconOnly.chromium.png|0| Added|
|Button.ButtonExampleInverted.chromium.png|0| Added|
|Button.ButtonExampleLoading.chromium.png|0| Added|
|Button.ButtonExampleOverflow.chromium.png|0| Added|
|Button.ButtonExampleRtl.chromium.png|0| Added|
|Button.ButtonExampleSizeShorthand.chromium.png|0| Added|
|Button.ButtonExampleTeams.chromium.png|0| Added|
|Button.ButtonExampleTeamsDark.chromium.png|0| Added|
|Button.ButtonExampleTeamsHighContrast.chromium.png|0| Added|
|Button.ButtonExampleText.chromium.png|0| Added|
|Button.ButtonExampleTextShorthand.chromium.png|0| Added|
|Button.ButtonExampleTextShorthandTeams.chromium.png|0| Added|
|Button.ButtonExampleTextShorthandTeamsDark.chromium.png|0| Added|
|Button.ButtonExampleTextShorthandTeamsHighContrast.chromium.png|0| Added|
|Button.ButtonExampleWithTooltip.chromium.png|0| Added|
|Button.ButtonExampleWithTooltipTeams.chromium.png|0| Added|
|Button.ButtonExampleWithTooltipTeamsDark.chromium.png|0| Added|
|Button.ButtonExampleWithTooltipTeamsHighContrast.chromium.png|0| Added|
|Button.ButtonGroupCircularExampleShorthand.chromium.png|0| Added|
|Button.ButtonGroupCircularExampleShorthandTeams.chromium.png|0| Added|
|Button.ButtonGroupCircularExampleShorthandTeamsDark.chromium.png|0| Added|
|Button.ButtonGroupCircularExampleShorthandTeamsHighContrast.chromium.png|0| Added|
|Button.ButtonGroupExampleShorthand.chromium.png|0| Added|
|Button.ButtonGroupExampleShorthandTeams.chromium.png|0| Added|
|Button.ButtonGroupExampleShorthandTeamsDark.chromium.png|0| Added|
|Button.ButtonGroupExampleShorthandTeamsHighContrast.chromium.png|0| Added|
|Button.ButtonUsageExampleShorthand.chromium.png|0| Added|
|Button.Button Default Bsize.chromium.png|0| Removed|
|Button.Button Example Circular.chromium.png|0| Removed|
|Button.Button Example Content And Icon Teams Dark.Clicks the first button.chromium.png|0| Removed|
|Button.Button Example Content And Icon Teams Dark.Focuses on the second button.chromium.png|0| Removed|
|Button.Button Example Content And Icon Teams Dark.Hovers the first button.chromium.png|0| Removed|
|Button.Button Example Content And Icon Teams Dark.chromium.png|0| Removed|
|Button.Button Example Content And Icon Teams High Contrast.Clicks the first button.chromium.png|0| Removed|
|Button.Button Example Content And Icon Teams High Contrast.Focuses on the second button.chromium.png|0| Removed|
|Button.Button Example Content And Icon Teams High Contrast.Hovers the first button.chromium.png|0| Removed|
|Button.Button Example Content And Icon Teams High Contrast.chromium.png|0| Removed|
|Button.Button Example Content And Icon Teams.Clicks the first button.chromium.png|0| Removed|
|Button.Button Example Content And Icon Teams.Focuses on the second button.chromium.png|0| Removed|
|Button.Button Example Content And Icon Teams.Hovers the first button.chromium.png|0| Removed|
|Button.Button Example Content And Icon Teams.chromium.png|0| Removed|
|Button.Button Example Content And Icon.Clicks the first button.chromium.png|0| Removed|
|Button.Button Example Content And Icon.Focuses on the second button.chromium.png|0| Removed|
|Button.Button Example Content And Icon.Hovers the first button.chromium.png|0| Removed|
|Button.Button Example Content And Icon.chromium.png|0| Removed|
|Button.Button Example Disabled Focusable.chromium.png|0| Removed|
|Button.Button Example Disabled Teams Dark.Clicks the first button.chromium.png|0| Removed|
|Button.Button Example Disabled Teams Dark.Focuses on the second button.chromium.png|0| Removed|
|Button.Button Example Disabled Teams Dark.Hovers the first button.chromium.png|0| Removed|
|Button.Button Example Disabled Teams Dark.chromium.png|0| Removed|
|Button.Button Example Disabled Teams High Contrast.Clicks the first button.chromium.png|0| Removed|
|Button.Button Example Disabled Teams High Contrast.Focuses on the second button.chromium.png|0| Removed|
|Button.Button Example Disabled Teams High Contrast.Hovers the first button.chromium.png|0| Removed|
|Button.Button Example Disabled Teams High Contrast.chromium.png|0| Removed|
|Button.Button Example Disabled Teams.Clicks the first button.chromium.png|0| Removed|
|Button.Button Example Disabled Teams.Focuses on the second button.chromium.png|0| Removed|
|Button.Button Example Disabled Teams.Hovers the first button.chromium.png|0| Removed|
|Button.Button Example Disabled Teams.chromium.png|0| Removed|
|Button.Button Example Disabled.Clicks the first button.chromium.png|0| Removed|
|Button.Button Example Disabled.Focuses on the second button.chromium.png|0| Removed|
|Button.Button Example Disabled.Hovers the first button.chromium.png|0| Removed|
|Button.Button Example Disabled.chromium.png|0| Removed|
|Button.Button Example Emphasis.chromium.png|0| Removed|
|Button.Button Example Flat.chromium.png|0| Removed|
|Button.Button Example Fluid.chromium.png|0| Removed|
|Button.Button Example Icon Only.chromium.png|0| Removed|
|Button.Button Example Inverted.chromium.png|0| Removed|
|Button.Button Example Loading.chromium.png|0| Removed|
|Button.Button Example Overflow.chromium.png|0| Removed|
|Button.Button Example Rtl.chromium.png|0| Removed|
|Button.Button Example Size Shorthand.chromium.png|0| Removed|
|Button.Button Example Teams Dark.Clicks the first button.chromium.png|0| Removed|
|Button.Button Example Teams Dark.Focuses on the second button.chromium.png|0| Removed|
|Button.Button Example Teams Dark.Hovers the first button.chromium.png|0| Removed|
|Button.Button Example Teams Dark.chromium.png|0| Removed|
|Button.Button Example Teams High Contrast.Clicks the first button.chromium.png|0| Removed|
|Button.Button Example Teams High Contrast.Focuses on the second button.chromium.png|0| Removed|
|Button.Button Example Teams High Contrast.Hovers the first button.chromium.png|0| Removed|
|Button.Button Example Teams High Contrast.chromium.png|0| Removed|
|Button.Button Example Teams.Clicks the first button.chromium.png|0| Removed|
|Button.Button Example Teams.Focuses on the second button.chromium.png|0| Removed|
|Button.Button Example Teams.Hovers the first button.chromium.png|0| Removed|
|Button.Button Example Teams.chromium.png|0| Removed|
|Button.Button Example Text Shorthand Teams Dark.Clicks the first button.chromium.png|0| Removed|
|Button.Button Example Text Shorthand Teams Dark.Focuses on the second button.chromium.png|0| Removed|
|Button.Button Example Text Shorthand Teams Dark.Hovers the first button.chromium.png|0| Removed|
|Button.Button Example Text Shorthand Teams Dark.chromium.png|0| Removed|
|Button.Button Example Text Shorthand Teams High Contrast.Clicks the first button.chromium.png|0| Removed|
|Button.Button Example Text Shorthand Teams High Contrast.Focuses on the second button.chromium.png|0| Removed|
|Button.Button Example Text Shorthand Teams High Contrast.Hovers the first button.chromium.png|0| Removed|
|Button.Button Example Text Shorthand Teams High Contrast.chromium.png|0| Removed|
|Button.Button Example Text Shorthand Teams.Clicks the first button.chromium.png|0| Removed|
|Button.Button Example Text Shorthand Teams.Focuses on the second button.chromium.png|0| Removed|
|Button.Button Example Text Shorthand Teams.Hovers the first button.chromium.png|0| Removed|
|Button.Button Example Text Shorthand Teams.chromium.png|0| Removed|
|Button.Button Example Text Shorthand.Clicks the first button.chromium.png|0| Removed|
|Button.Button Example Text Shorthand.Focuses on the second button.chromium.png|0| Removed|
|Button.Button Example Text Shorthand.Hovers the first button.chromium.png|0| Removed|
|Button.Button Example Text Shorthand.chromium.png|0| Removed|
|Button.Button Example Text.chromium.png|0| Removed|
|Button.Button Example With Tooltip Teams Dark.Clicks the first button.chromium.png|0| Removed|
|Button.Button Example With Tooltip Teams Dark.Focuses on the second button.chromium.png|0| Removed|
|Button.Button Example With Tooltip Teams Dark.Hovers the first button.chromium.png|0| Removed|
|Button.Button Example With Tooltip Teams Dark.chromium.png|0| Removed|
|Button.Button Example With Tooltip Teams High Contrast.Clicks the first button.chromium.png|0| Removed|
|Button.Button Example With Tooltip Teams High Contrast.Focuses on the second button.chromium.png|0| Removed|
|Button.Button Example With Tooltip Teams High Contrast.Hovers the first button.chromium.png|0| Removed|
|Button.Button Example With Tooltip Teams High Contrast.chromium.png|0| Removed|
|Button.Button Example With Tooltip Teams.Clicks the first button.chromium.png|0| Removed|
|Button.Button Example With Tooltip Teams.Focuses on the second button.chromium.png|0| Removed|
|Button.Button Example With Tooltip Teams.Hovers the first button.chromium.png|0| Removed|
|Button.Button Example With Tooltip Teams.chromium.png|0| Removed|
|Button.Button Example With Tooltip.Clicks the first button.chromium.png|0| Removed|
|Button.Button Example With Tooltip.Focuses on the second button.chromium.png|0| Removed|
|Button.Button Example With Tooltip.Hovers the first button.chromium.png|0| Removed|
|Button.Button Example With Tooltip.chromium.png|0| Removed|
|Button.Button Example.Clicks the first button.chromium.png|0| Removed|
|Button.Button Example.Focuses on the second button.chromium.png|0| Removed|
|Button.Button Example.Hovers the first button.chromium.png|0| Removed|
|Button.Button Example.chromium.png|0| Removed|
|Button.Button Group Circular Example Shorthand Teams Dark.Clicks the first button.chromium.png|0| Removed|
|Button.Button Group Circular Example Shorthand Teams Dark.Focuses on the second button.chromium.png|0| Removed|
|Button.Button Group Circular Example Shorthand Teams Dark.Hovers the first button.chromium.png|0| Removed|
|Button.Button Group Circular Example Shorthand Teams Dark.chromium.png|0| Removed|
|Button.Button Group Circular Example Shorthand Teams High Contrast.Clicks the first button.chromium.png|0| Removed|
|Button.Button Group Circular Example Shorthand Teams High Contrast.Focuses on the second button.chromium.png|0| Removed|
|Button.Button Group Circular Example Shorthand Teams High Contrast.Hovers the first button.chromium.png|0| Removed|
|Button.Button Group Circular Example Shorthand Teams High Contrast.chromium.png|0| Removed|
|Button.Button Group Circular Example Shorthand Teams.Clicks the first button.chromium.png|0| Removed|
|Button.Button Group Circular Example Shorthand Teams.Focuses on the second button.chromium.png|0| Removed|
|Button.Button Group Circular Example Shorthand Teams.Hovers the first button.chromium.png|0| Removed|
|Button.Button Group Circular Example Shorthand Teams.chromium.png|0| Removed|
|Button.Button Group Circular Example Shorthand.Clicks the first button.chromium.png|0| Removed|
|Button.Button Group Circular Example Shorthand.Focuses on the second button.chromium.png|0| Removed|
|Button.Button Group Circular Example Shorthand.Hovers the first button.chromium.png|0| Removed|
|Button.Button Group Circular Example Shorthand.chromium.png|0| Removed|
|Button.Button Group Example Shorthand Teams Dark.Clicks the first button.chromium.png|0| Removed|
|Button.Button Group Example Shorthand Teams Dark.Focuses on the second button.chromium.png|0| Removed|
|Button.Button Group Example Shorthand Teams Dark.Hovers the first button.chromium.png|0| Removed|
|Button.Button Group Example Shorthand Teams Dark.chromium.png|0| Removed|
|Button.Button Group Example Shorthand Teams High Contrast.Clicks the first button.chromium.png|0| Removed|
|Button.Button Group Example Shorthand Teams High Contrast.Focuses on the second button.chromium.png|0| Removed|
|Button.Button Group Example Shorthand Teams High Contrast.Hovers the first button.chromium.png|0| Removed|
|Button.Button Group Example Shorthand Teams High Contrast.chromium.png|0| Removed|
|Button.Button Group Example Shorthand Teams.Clicks the first button.chromium.png|0| Removed|
|Button.Button Group Example Shorthand Teams.Focuses on the second button.chromium.png|0| Removed|
|Button.Button Group Example Shorthand Teams.Hovers the first button.chromium.png|0| Removed|
|Button.Button Group Example Shorthand Teams.chromium.png|0| Removed|
|Button.Button Group Example Shorthand.Clicks the first button.chromium.png|0| Removed|
|Button.Button Group Example Shorthand.Focuses on the second button.chromium.png|0| Removed|
|Button.Button Group Example Shorthand.Hovers the first button.chromium.png|0| Removed|
|Button.Button Group Example Shorthand.chromium.png|0| Removed|
|Button.Button Usage Example Shorthand.chromium.png|0| Removed|

</details>


  <details><summary><b>Card</b> 52 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card.CardExampleBody.chromium.png|0| Added|
|Card.CardExampleCentered.chromium.png|0| Added|
|Card.CardExampleContextMenu.chromium.png|0| Added|
|Card.CardExampleDisabled.chromium.png|0| Added|
|Card.CardExampleElevated.chromium.png|0| Added|
|Card.CardExampleExpandable.chromium.png|0| Added|
|Card.CardExampleFocusable.chromium.png|0| Added|
|Card.CardExampleFocusableGrid.chromium.png|0| Added|
|Card.CardExampleFocusableTeams.chromium.png|0| Added|
|Card.CardExampleFocusableTeamsDark.chromium.png|0| Added|
|Card.CardExampleFocusableTeamsHighContrast.chromium.png|0| Added|
|Card.CardExampleFooter.chromium.png|0| Added|
|Card.CardExampleHeader.chromium.png|0| Added|
|Card.CardExampleHorizontal.chromium.png|0| Added|
|Card.CardExampleInverted.chromium.png|0| Added|
|Card.CardExamplePreview.chromium.png|0| Added|
|Card.CardExampleQuiet.chromium.png|0| Added|
|Card.CardExampleSelectableGrid.chromium.png|0| Added|
|Card.CardExampleSelected.chromium.png|0| Added|
|Card.CardExampleSimple.chromium.png|0| Added|
|Card.CardExampleSize.chromium.png|0| Added|
|Card.CardExampleWithPreview.chromium.png|0| Added|
|Card.CardExampleWithPreviewTeams.chromium.png|0| Added|
|Card.CardExampleWithPreviewTeamsDark.chromium.png|0| Added|
|Card.CardExampleWithPreviewTeamsHighContrast.chromium.png|0| Added|
|Card.ImageExampleFluent.chromium.png|0| Added|
|Card.Card Example Body.chromium.png|0| Removed|
|Card.Card Example Centered.chromium.png|0| Removed|
|Card.Card Example Context Menu.chromium.png|0| Removed|
|Card.Card Example Disabled.chromium.png|0| Removed|
|Card.Card Example Elevated.chromium.png|0| Removed|
|Card.Card Example Expandable.chromium.png|0| Removed|
|Card.Card Example Focusable Grid.chromium.png|0| Removed|
|Card.Card Example Focusable Teams Dark.chromium.png|0| Removed|
|Card.Card Example Focusable Teams High Contrast.chromium.png|0| Removed|
|Card.Card Example Focusable Teams.chromium.png|0| Removed|
|Card.Card Example Focusable.chromium.png|0| Removed|
|Card.Card Example Footer.chromium.png|0| Removed|
|Card.Card Example Header.chromium.png|0| Removed|
|Card.Card Example Horizontal.chromium.png|0| Removed|
|Card.Card Example Inverted.chromium.png|0| Removed|
|Card.Card Example Preview.chromium.png|0| Removed|
|Card.Card Example Quiet.chromium.png|0| Removed|
|Card.Card Example Selectable Grid.chromium.png|0| Removed|
|Card.Card Example Selected.chromium.png|0| Removed|
|Card.Card Example Simple.chromium.png|0| Removed|
|Card.Card Example Size.chromium.png|0| Removed|
|Card.Card Example With Preview Teams Dark.chromium.png|0| Removed|
|Card.Card Example With Preview Teams High Contrast.chromium.png|0| Removed|
|Card.Card Example With Preview Teams.chromium.png|0| Removed|
|Card.Card Example With Preview.chromium.png|0| Removed|
|Card.Image Example Fluent.chromium.png|0| Removed|

</details>


  <details><summary><b>Carousel</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Carousel.CarouselExample.chromium.png|0| Added|
|Carousel.CarouselSlideAnimationExample.chromium.png|0| Added|
|Carousel.Carousel Example.chromium.png|0| Removed|
|Carousel.Carousel Slide Animation Example.chromium.png|0| Removed|

</details>


  <details><summary><b>Chat</b> 58 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Chat.ChatExample.chromium.png|0| Added|
|Chat.ChatExampleComfyRefresh.chromium.png|0| Added|
|Chat.ChatExampleCompact.chromium.png|0| Added|
|Chat.ChatExampleCompactTeams.chromium.png|0| Added|
|Chat.ChatExampleCompactTeamsDark.chromium.png|0| Added|
|Chat.ChatExampleCompactTeamsHighContrast.chromium.png|0| Added|
|Chat.ChatExampleContentPosition.chromium.png|0| Added|
|Chat.ChatExampleDetails.chromium.png|0| Added|
|Chat.ChatExampleDetailsTeams.chromium.png|0| Added|
|Chat.ChatExampleDetailsTeamsDark.chromium.png|0| Added|
|Chat.ChatExampleDetailsTeamsHighContrast.chromium.png|0| Added|
|Chat.ChatExampleHeader.chromium.png|0| Added|
|Chat.ChatExampleHeaderOverride.chromium.png|0| Added|
|Chat.ChatExampleInScrollableShorthand.chromium.png|0| Added|
|Chat.ChatExampleReactionGroup.chromium.png|0| Added|
|Chat.ChatExampleReadStatus.chromium.png|0| Added|
|Chat.ChatExampleRtl.chromium.png|0| Added|
|Chat.ChatExampleTeams.chromium.png|0| Added|
|Chat.ChatExampleTeamsDark.chromium.png|0| Added|
|Chat.ChatExampleTeamsHighContrast.chromium.png|0| Added|
|Chat.ChatMessageExampleBadge.chromium.png|0| Added|
|Chat.ChatMessageExampleBadgeTeams.chromium.png|0| Added|
|Chat.ChatMessageExampleBadgeTeamsDark.chromium.png|0| Added|
|Chat.ChatMessageExampleBadgeTeamsHighContrast.chromium.png|0| Added|
|Chat.ChatMessageExampleStyled.chromium.png|0| Added|
|Chat.MessageReactionsWithPopup.chromium.png|0| Added|
|Chat.MessageReactionsWithPopupTeams.chromium.png|0| Added|
|Chat.MessageReactionsWithPopupTeamsDark.chromium.png|0| Added|
|Chat.MessageReactionsWithPopupTeamsHighContrast.chromium.png|0| Added|
|Chat.Chat Example Comfy Refresh.chromium.png|0| Removed|
|Chat.Chat Example Compact Teams Dark.chromium.png|0| Removed|
|Chat.Chat Example Compact Teams High Contrast.chromium.png|0| Removed|
|Chat.Chat Example Compact Teams.chromium.png|0| Removed|
|Chat.Chat Example Compact.chromium.png|0| Removed|
|Chat.Chat Example Content Position.chromium.png|0| Removed|
|Chat.Chat Example Details Teams Dark.chromium.png|0| Removed|
|Chat.Chat Example Details Teams High Contrast.chromium.png|0| Removed|
|Chat.Chat Example Details Teams.chromium.png|0| Removed|
|Chat.Chat Example Details.chromium.png|0| Removed|
|Chat.Chat Example Header Override.chromium.png|0| Removed|
|Chat.Chat Example Header.chromium.png|0| Removed|
|Chat.Chat Example In Scrollable Shorthand.chromium.png|0| Removed|
|Chat.Chat Example Reaction Group.chromium.png|0| Removed|
|Chat.Chat Example Read Status.chromium.png|0| Removed|
|Chat.Chat Example Rtl.chromium.png|0| Removed|
|Chat.Chat Example Teams Dark.chromium.png|0| Removed|
|Chat.Chat Example Teams High Contrast.chromium.png|0| Removed|
|Chat.Chat Example Teams.chromium.png|0| Removed|
|Chat.Chat Example.chromium.png|0| Removed|
|Chat.Chat Message Example Badge Teams Dark.chromium.png|0| Removed|
|Chat.Chat Message Example Badge Teams High Contrast.chromium.png|0| Removed|
|Chat.Chat Message Example Badge Teams.chromium.png|0| Removed|
|Chat.Chat Message Example Badge.chromium.png|0| Removed|
|Chat.Chat Message Example Styled.chromium.png|0| Removed|
|Chat.Message Reactions With Popup Teams Dark.chromium.png|0| Removed|
|Chat.Message Reactions With Popup Teams High Contrast.chromium.png|0| Removed|
|Chat.Message Reactions With Popup Teams.chromium.png|0| Removed|
|Chat.Message Reactions With Popup.chromium.png|0| Removed|

</details>


  <details><summary><b>Checkbox</b> 26 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Checkbox.CheckboxExample.chromium.png|0| Added|
|Checkbox.CheckboxExampleChecked.chromium.png|0| Added|
|Checkbox.CheckboxExampleCheckedMixed.chromium.png|0| Added|
|Checkbox.CheckboxExampleCheckedMixedTeams.chromium.png|0| Added|
|Checkbox.CheckboxExampleCheckedMixedTeamsDark.chromium.png|0| Added|
|Checkbox.CheckboxExampleCheckedMixedTeamsHighContrast.chromium.png|0| Added|
|Checkbox.CheckboxExampleDisabled.chromium.png|0| Added|
|Checkbox.CheckboxExampleLabel.chromium.png|0| Added|
|Checkbox.CheckboxExampleLabelFlexColumn.chromium.png|0| Added|
|Checkbox.CheckboxExampleRtl.chromium.png|0| Added|
|Checkbox.CheckboxExampleToggle.chromium.png|0| Added|
|Checkbox.Checkbox Example Checked Mixed Teams Dark.chromium.png|0| Removed|
|Checkbox.Checkbox Example Checked Mixed Teams High Contrast.chromium.png|0| Removed|
|Checkbox.Checkbox Example Checked Mixed Teams.chromium.png|0| Removed|
|Checkbox.Checkbox Example Checked Mixed.chromium.png|0| Removed|
|Checkbox.Checkbox Example Checked.chromium.png|0| Removed|
|Checkbox.Checkbox Example Disabled.chromium.png|0| Removed|
|Checkbox.Checkbox Example Label Flex Column.chromium.png|0| Removed|
|Checkbox.Checkbox Example Label.chromium.png|0| Removed|
|Checkbox.Checkbox Example Rtl.chromium.png|0| Removed|
|Checkbox.Checkbox Example Toggle.Checks checkbox.chromium.png|0| Removed|
|Checkbox.Checkbox Example Toggle.Focuses checkbox.chromium.png|0| Removed|
|Checkbox.Checkbox Example Toggle.chromium.png|0| Removed|
|Checkbox.Checkbox Example.Checks checkbox.chromium.png|0| Removed|
|Checkbox.Checkbox Example.Focuses checkbox.chromium.png|0| Removed|
|Checkbox.Checkbox Example.chromium.png|0| Removed|

</details>


  <details><summary><b>Datepicker</b> 222 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Datepicker.DatepickerCellExample.chromium.png|0| Added|
|Datepicker.DatepickerCellExampleTeams.chromium.png|0| Added|
|Datepicker.DatepickerCellExampleTeamsDark.chromium.png|0| Added|
|Datepicker.DatepickerCellExampleTeamsHighContrast.chromium.png|0| Added|
|Datepicker.DatepickerExample.chromium.png|0| Added|
|Datepicker.DatepickerExampleClearable.chromium.png|0| Added|
|Datepicker.DatepickerExampleClearableTeams.chromium.png|0| Added|
|Datepicker.DatepickerExampleClearableTeamsDark.chromium.png|0| Added|
|Datepicker.DatepickerExampleClearableTeamsHighContrast.chromium.png|0| Added|
|Datepicker.DatepickerExampleDisabled.chromium.png|0| Added|
|Datepicker.DatepickerExampleLocalizationStrings.chromium.png|0| Added|
|Datepicker.DatepickerExampleLocalizationStringsTeams.chromium.png|0| Added|
|Datepicker.DatepickerExampleLocalizationStringsTeamsDark.chromium.png|0| Added|
|Datepicker.DatepickerExampleLocalizationStringsTeamsHighContrast.chromium.png|0| Added|
|Datepicker.DatepickerExampleMinMaxDate.chromium.png|0| Added|
|Datepicker.DatepickerExampleMinMaxDateTeams.chromium.png|0| Added|
|Datepicker.DatepickerExampleMinMaxDateTeamsDark.chromium.png|0| Added|
|Datepicker.DatepickerExampleMinMaxDateTeamsHighContrast.chromium.png|0| Added|
|Datepicker.DatepickerExampleNoInput.chromium.png|0| Added|
|Datepicker.DatepickerExampleNoInputTeams.chromium.png|0| Added|
|Datepicker.DatepickerExampleNoInputTeamsDark.chromium.png|0| Added|
|Datepicker.DatepickerExampleNoInputTeamsHighContrast.chromium.png|0| Added|
|Datepicker.DatepickerExampleOpen.chromium.png|0| Added|
|Datepicker.DatepickerExampleRequired.chromium.png|0| Added|
|Datepicker.DatepickerExampleRequiredTeams.chromium.png|0| Added|
|Datepicker.DatepickerExampleRequiredTeamsDark.chromium.png|0| Added|
|Datepicker.DatepickerExampleRequiredTeamsHighContrast.chromium.png|0| Added|
|Datepicker.DatepickerExampleRestrictedDates.chromium.png|0| Added|
|Datepicker.DatepickerExampleRestrictedDatesTeams.chromium.png|0| Added|
|Datepicker.DatepickerExampleRestrictedDatesTeamsDark.chromium.png|0| Added|
|Datepicker.DatepickerExampleRestrictedDatesTeamsHighContrast.chromium.png|0| Added|
|Datepicker.DatepickerExampleSelectedDate.chromium.png|0| Added|
|Datepicker.DatepickerExampleStandaloneCalendarButton.chromium.png|0| Added|
|Datepicker.DatepickerExampleStandaloneCalendarButtonTeams.chromium.png|0| Added|
|Datepicker.DatepickerExampleStandaloneCalendarButtonTeamsDark.chromium.png|0| Added|
|Datepicker.DatepickerExampleStandaloneCalendarButtonTeamsHighContrast.chromium.png|0| Added|
|Datepicker.DatepickerExampleStandaloneCalendarInput.chromium.png|0| Added|
|Datepicker.DatepickerExampleToday.chromium.png|0| Added|
|Datepicker.DatepickerExampleTodayTeams.chromium.png|0| Added|
|Datepicker.DatepickerExampleTodayTeamsDark.chromium.png|0| Added|
|Datepicker.DatepickerExampleTodayTeamsHighContrast.chromium.png|0| Added|
|Datepicker.DatepickerExampleWeek.chromium.png|0| Added|
|Datepicker.DatepickerExampleWeekTeams.chromium.png|0| Added|
|Datepicker.DatepickerExampleWeekTeamsDark.chromium.png|0| Added|
|Datepicker.DatepickerExampleWeekTeamsHighContrast.chromium.png|0| Added|
|Datepicker.DatepickerFirstWeekDayExample.chromium.png|0| Added|
|Datepicker.DatepickerFormatExample.chromium.png|0| Added|
|Datepicker.DatepickerFormatExampleTeams.chromium.png|0| Added|
|Datepicker.DatepickerFormatExampleTeamsDark.chromium.png|0| Added|
|Datepicker.DatepickerFormatExampleTeamsHighContrast.chromium.png|0| Added|
|Datepicker.DatepickerHeaderCellExample.chromium.png|0| Added|
|Datepicker.DatepickerHeaderCellExampleTeams.chromium.png|0| Added|
|Datepicker.DatepickerHeaderCellExampleTeamsDark.chromium.png|0| Added|
|Datepicker.DatepickerHeaderCellExampleTeamsHighContrast.chromium.png|0| Added|
|Datepicker.DatepickerHeaderExample.chromium.png|0| Added|
|Datepicker.DatepickerNoInputPlaceholderExample.chromium.png|0| Added|
|Datepicker.DatepickerParseExample.chromium.png|0| Added|
|Datepicker.Datepicker Cell Example Teams Dark.Does not show tooltip on not today's date..chromium.png|0| Removed|
|Datepicker.Datepicker Cell Example Teams Dark.Shows datepicker popup..chromium.png|0| Removed|
|Datepicker.Datepicker Cell Example Teams Dark.Shows tooltip on today's date..chromium.png|0| Removed|
|Datepicker.Datepicker Cell Example Teams Dark.chromium.png|0| Removed|
|Datepicker.Datepicker Cell Example Teams High Contrast.Does not show tooltip on not today's date..chromium.png|0| Removed|
|Datepicker.Datepicker Cell Example Teams High Contrast.Shows datepicker popup..chromium.png|0| Removed|
|Datepicker.Datepicker Cell Example Teams High Contrast.Shows tooltip on today's date..chromium.png|0| Removed|
|Datepicker.Datepicker Cell Example Teams High Contrast.chromium.png|0| Removed|
|Datepicker.Datepicker Cell Example Teams.Does not show tooltip on not today's date..chromium.png|0| Removed|
|Datepicker.Datepicker Cell Example Teams.Shows datepicker popup..chromium.png|0| Removed|
|Datepicker.Datepicker Cell Example Teams.Shows tooltip on today's date..chromium.png|0| Removed|
|Datepicker.Datepicker Cell Example Teams.chromium.png|0| Removed|
|Datepicker.Datepicker Cell Example.Does not show tooltip on not today's date..chromium.png|0| Removed|
|Datepicker.Datepicker Cell Example.Shows datepicker popup..chromium.png|0| Removed|
|Datepicker.Datepicker Cell Example.Shows tooltip on today's date..chromium.png|0| Removed|
|Datepicker.Datepicker Cell Example.chromium.png|0| Removed|
|Datepicker.Datepicker Example Clearable Teams Dark.Shows cleared calendar..chromium.png|0| Removed|
|Datepicker.Datepicker Example Clearable Teams Dark.Shows cleared input..chromium.png|0| Removed|
|Datepicker.Datepicker Example Clearable Teams Dark.Shows selected date in input with clear possibility..chromium.png|0| Removed|
|Datepicker.Datepicker Example Clearable Teams Dark.chromium.png|0| Removed|
|Datepicker.Datepicker Example Clearable Teams High Contrast.Shows cleared calendar..chromium.png|0| Removed|
|Datepicker.Datepicker Example Clearable Teams High Contrast.Shows cleared input..chromium.png|0| Removed|
|Datepicker.Datepicker Example Clearable Teams High Contrast.Shows selected date in input with clear possibility..chromium.png|0| Removed|
|Datepicker.Datepicker Example Clearable Teams High Contrast.chromium.png|0| Removed|
|Datepicker.Datepicker Example Clearable Teams.Shows cleared calendar..chromium.png|0| Removed|
|Datepicker.Datepicker Example Clearable Teams.Shows cleared input..chromium.png|0| Removed|
|Datepicker.Datepicker Example Clearable Teams.Shows selected date in input with clear possibility..chromium.png|0| Removed|
|Datepicker.Datepicker Example Clearable Teams.chromium.png|0| Removed|
|Datepicker.Datepicker Example Clearable.Shows cleared calendar..chromium.png|0| Removed|
|Datepicker.Datepicker Example Clearable.Shows cleared input..chromium.png|0| Removed|
|Datepicker.Datepicker Example Clearable.Shows selected date in input with clear possibility..chromium.png|0| Removed|
|Datepicker.Datepicker Example Clearable.chromium.png|0| Removed|
|Datepicker.Datepicker Example Disabled.chromium.png|0| Removed|
|Datepicker.Datepicker Example Localization Strings Teams Dark.Shows localized calendar..chromium.png|0| Removed|
|Datepicker.Datepicker Example Localization Strings Teams Dark.chromium.png|0| Removed|
|Datepicker.Datepicker Example Localization Strings Teams High Contrast.Shows localized calendar..chromium.png|0| Removed|
|Datepicker.Datepicker Example Localization Strings Teams High Contrast.chromium.png|0| Removed|
|Datepicker.Datepicker Example Localization Strings Teams.Shows localized calendar..chromium.png|0| Removed|
|Datepicker.Datepicker Example Localization Strings Teams.chromium.png|0| Removed|
|Datepicker.Datepicker Example Localization Strings.Shows localized calendar..chromium.png|0| Removed|
|Datepicker.Datepicker Example Localization Strings.chromium.png|0| Removed|
|Datepicker.Datepicker Example Min Max Date Teams Dark.Disabled date is not clickable..chromium.png|0| Removed|
|Datepicker.Datepicker Example Min Max Date Teams Dark.Month icon is not clickable..chromium.png|0| Removed|
|Datepicker.Datepicker Example Min Max Date Teams Dark.Shows min max dates..chromium.png|0| Removed|
|Datepicker.Datepicker Example Min Max Date Teams Dark.chromium.png|0| Removed|
|Datepicker.Datepicker Example Min Max Date Teams High Contrast.Disabled date is not clickable..chromium.png|0| Removed|
|Datepicker.Datepicker Example Min Max Date Teams High Contrast.Month icon is not clickable..chromium.png|0| Removed|
|Datepicker.Datepicker Example Min Max Date Teams High Contrast.Shows min max dates..chromium.png|0| Removed|
|Datepicker.Datepicker Example Min Max Date Teams High Contrast.chromium.png|0| Removed|
|Datepicker.Datepicker Example Min Max Date Teams.Disabled date is not clickable..chromium.png|0| Removed|
|Datepicker.Datepicker Example Min Max Date Teams.Month icon is not clickable..chromium.png|0| Removed|
|Datepicker.Datepicker Example Min Max Date Teams.Shows min max dates..chromium.png|0| Removed|
|Datepicker.Datepicker Example Min Max Date Teams.chromium.png|0| Removed|
|Datepicker.Datepicker Example Min Max Date.Disabled date is not clickable..chromium.png|0| Removed|
|Datepicker.Datepicker Example Min Max Date.Month icon is not clickable..chromium.png|0| Removed|
|Datepicker.Datepicker Example Min Max Date.Shows min max dates..chromium.png|0| Removed|
|Datepicker.Datepicker Example Min Max Date.chromium.png|0| Removed|
|Datepicker.Datepicker Example No Input Teams Dark.Keeps calendar open..chromium.png|0| Removed|
|Datepicker.Datepicker Example No Input Teams Dark.Shows calendar..chromium.png|0| Removed|
|Datepicker.Datepicker Example No Input Teams Dark.chromium.png|0| Removed|
|Datepicker.Datepicker Example No Input Teams High Contrast.Keeps calendar open..chromium.png|0| Removed|
|Datepicker.Datepicker Example No Input Teams High Contrast.Shows calendar..chromium.png|0| Removed|
|Datepicker.Datepicker Example No Input Teams High Contrast.chromium.png|0| Removed|
|Datepicker.Datepicker Example No Input Teams.Keeps calendar open..chromium.png|0| Removed|
|Datepicker.Datepicker Example No Input Teams.Shows calendar..chromium.png|0| Removed|
|Datepicker.Datepicker Example No Input Teams.chromium.png|0| Removed|
|Datepicker.Datepicker Example No Input.Keeps calendar open..chromium.png|0| Removed|
|Datepicker.Datepicker Example No Input.Shows calendar..chromium.png|0| Removed|
|Datepicker.Datepicker Example No Input.chromium.png|0| Removed|
|Datepicker.Datepicker Example Open.chromium.png|0| Removed|
|Datepicker.Datepicker Example Required Teams Dark.Input shows error..chromium.png|0| Removed|
|Datepicker.Datepicker Example Required Teams Dark.Shows datepicker popup through button and error is shown..chromium.png|0| Removed|
|Datepicker.Datepicker Example Required Teams Dark.Shows selected date in input without error..chromium.png|0| Removed|
|Datepicker.Datepicker Example Required Teams High Contrast.Input shows error..chromium.png|0| Removed|
|Datepicker.Datepicker Example Required Teams High Contrast.Shows datepicker popup through button and error is shown..chromium.png|0| Removed|
|Datepicker.Datepicker Example Required Teams High Contrast.Shows selected date in input without error..chromium.png|0| Removed|
|Datepicker.Datepicker Example Required Teams.Input shows error..chromium.png|0| Removed|
|Datepicker.Datepicker Example Required Teams.Shows datepicker popup through button and error is shown..chromium.png|0| Removed|
|Datepicker.Datepicker Example Required Teams.Shows selected date in input without error..chromium.png|0| Removed|
|Datepicker.Datepicker Example Required.Input shows error..chromium.png|0| Removed|
|Datepicker.Datepicker Example Required.Shows datepicker popup through button and error is shown..chromium.png|0| Removed|
|Datepicker.Datepicker Example Required.Shows selected date in input without error..chromium.png|0| Removed|
|Datepicker.Datepicker Example Restricted Dates Teams Dark.Shows disabled dates..chromium.png|0| Removed|
|Datepicker.Datepicker Example Restricted Dates Teams Dark.chromium.png|0| Removed|
|Datepicker.Datepicker Example Restricted Dates Teams High Contrast.Shows disabled dates..chromium.png|0| Removed|
|Datepicker.Datepicker Example Restricted Dates Teams High Contrast.chromium.png|0| Removed|
|Datepicker.Datepicker Example Restricted Dates Teams.Shows disabled dates..chromium.png|0| Removed|
|Datepicker.Datepicker Example Restricted Dates Teams.chromium.png|0| Removed|
|Datepicker.Datepicker Example Restricted Dates.Shows disabled dates..chromium.png|0| Removed|
|Datepicker.Datepicker Example Restricted Dates.chromium.png|0| Removed|
|Datepicker.Datepicker Example Selected Date.chromium.png|0| Removed|
|Datepicker.Datepicker Example Standalone Calendar Button Teams Dark.Shows datepicker calendar..chromium.png|0| Removed|
|Datepicker.Datepicker Example Standalone Calendar Button Teams Dark.chromium.png|0| Removed|
|Datepicker.Datepicker Example Standalone Calendar Button Teams High Contrast.Shows datepicker calendar..chromium.png|0| Removed|
|Datepicker.Datepicker Example Standalone Calendar Button Teams High Contrast.chromium.png|0| Removed|
|Datepicker.Datepicker Example Standalone Calendar Button Teams.Shows datepicker calendar..chromium.png|0| Removed|
|Datepicker.Datepicker Example Standalone Calendar Button Teams.chromium.png|0| Removed|
|Datepicker.Datepicker Example Standalone Calendar Button.Shows datepicker calendar..chromium.png|0| Removed|
|Datepicker.Datepicker Example Standalone Calendar Button.chromium.png|0| Removed|
|Datepicker.Datepicker Example Standalone Calendar Input.chromium.png|0| Removed|
|Datepicker.Datepicker Example Today Teams Dark.Shows calendar with hover..chromium.png|0| Removed|
|Datepicker.Datepicker Example Today Teams Dark.Shows datepicker popup through button..chromium.png|0| Removed|
|Datepicker.Datepicker Example Today Teams Dark.Shows selected date in calendar..chromium.png|0| Removed|
|Datepicker.Datepicker Example Today Teams Dark.Shows selected date in input..chromium.png|0| Removed|
|Datepicker.Datepicker Example Today Teams Dark.chromium.png|0| Removed|
|Datepicker.Datepicker Example Today Teams High Contrast.Shows calendar with hover..chromium.png|0| Removed|
|Datepicker.Datepicker Example Today Teams High Contrast.Shows datepicker popup through button..chromium.png|0| Removed|
|Datepicker.Datepicker Example Today Teams High Contrast.Shows selected date in calendar..chromium.png|0| Removed|
|Datepicker.Datepicker Example Today Teams High Contrast.Shows selected date in input..chromium.png|0| Removed|
|Datepicker.Datepicker Example Today Teams High Contrast.chromium.png|0| Removed|
|Datepicker.Datepicker Example Today Teams.Shows calendar with hover..chromium.png|0| Removed|
|Datepicker.Datepicker Example Today Teams.Shows datepicker popup through button..chromium.png|0| Removed|
|Datepicker.Datepicker Example Today Teams.Shows selected date in calendar..chromium.png|0| Removed|
|Datepicker.Datepicker Example Today Teams.Shows selected date in input..chromium.png|0| Removed|
|Datepicker.Datepicker Example Today Teams.chromium.png|0| Removed|
|Datepicker.Datepicker Example Today.Shows calendar with hover..chromium.png|0| Removed|
|Datepicker.Datepicker Example Today.Shows datepicker popup through button..chromium.png|0| Removed|
|Datepicker.Datepicker Example Today.Shows selected date in calendar..chromium.png|0| Removed|
|Datepicker.Datepicker Example Today.Shows selected date in input..chromium.png|0| Removed|
|Datepicker.Datepicker Example Today.chromium.png|0| Removed|
|Datepicker.Datepicker Example Week Teams Dark.Calendar is opened with selected week highlighted..chromium.png|0| Removed|
|Datepicker.Datepicker Example Week Teams Dark.Calendar is opened with the entire week highlighted on hover..chromium.png|0| Removed|
|Datepicker.Datepicker Example Week Teams Dark.Shows calendar..chromium.png|0| Removed|
|Datepicker.Datepicker Example Week Teams Dark.chromium.png|0| Removed|
|Datepicker.Datepicker Example Week Teams High Contrast.Calendar is opened with selected week highlighted..chromium.png|0| Removed|
|Datepicker.Datepicker Example Week Teams High Contrast.Calendar is opened with the entire week highlighted on hover..chromium.png|0| Removed|
|Datepicker.Datepicker Example Week Teams High Contrast.Shows calendar..chromium.png|0| Removed|
|Datepicker.Datepicker Example Week Teams High Contrast.chromium.png|0| Removed|
|Datepicker.Datepicker Example Week Teams.Calendar is opened with selected week highlighted..chromium.png|0| Removed|
|Datepicker.Datepicker Example Week Teams.Calendar is opened with the entire week highlighted on hover..chromium.png|0| Removed|
|Datepicker.Datepicker Example Week Teams.Shows calendar..chromium.png|0| Removed|
|Datepicker.Datepicker Example Week Teams.chromium.png|0| Removed|
|Datepicker.Datepicker Example Week.Calendar is opened with selected week highlighted..chromium.png|0| Removed|
|Datepicker.Datepicker Example Week.Calendar is opened with the entire week highlighted on hover..chromium.png|0| Removed|
|Datepicker.Datepicker Example Week.Shows calendar..chromium.png|0| Removed|
|Datepicker.Datepicker Example Week.chromium.png|0| Removed|
|Datepicker.Datepicker Example.chromium.png|0| Removed|
|Datepicker.Datepicker First Week Day Example.chromium.png|0| Removed|
|Datepicker.Datepicker Format Example Teams Dark.Shows datepicker popup.chromium.png|0| Removed|
|Datepicker.Datepicker Format Example Teams Dark.Shows selected date.chromium.png|0| Removed|
|Datepicker.Datepicker Format Example Teams Dark.chromium.png|0| Removed|
|Datepicker.Datepicker Format Example Teams High Contrast.Shows datepicker popup.chromium.png|0| Removed|
|Datepicker.Datepicker Format Example Teams High Contrast.Shows selected date.chromium.png|0| Removed|
|Datepicker.Datepicker Format Example Teams High Contrast.chromium.png|0| Removed|
|Datepicker.Datepicker Format Example Teams.Shows datepicker popup.chromium.png|0| Removed|
|Datepicker.Datepicker Format Example Teams.Shows selected date.chromium.png|0| Removed|
|Datepicker.Datepicker Format Example Teams.chromium.png|0| Removed|
|Datepicker.Datepicker Format Example.Shows datepicker popup.chromium.png|0| Removed|
|Datepicker.Datepicker Format Example.Shows selected date.chromium.png|0| Removed|
|Datepicker.Datepicker Format Example.chromium.png|0| Removed|
|Datepicker.Datepicker Header Cell Example Teams Dark.Shows datepicker popup..chromium.png|0| Removed|
|Datepicker.Datepicker Header Cell Example Teams Dark.Shows tooltip on today's date..chromium.png|0| Removed|
|Datepicker.Datepicker Header Cell Example Teams Dark.chromium.png|0| Removed|
|Datepicker.Datepicker Header Cell Example Teams High Contrast.Shows datepicker popup..chromium.png|0| Removed|
|Datepicker.Datepicker Header Cell Example Teams High Contrast.Shows tooltip on today's date..chromium.png|0| Removed|
|Datepicker.Datepicker Header Cell Example Teams High Contrast.chromium.png|0| Removed|
|Datepicker.Datepicker Header Cell Example Teams.Shows datepicker popup..chromium.png|0| Removed|
|Datepicker.Datepicker Header Cell Example Teams.Shows tooltip on today's date..chromium.png|0| Removed|
|Datepicker.Datepicker Header Cell Example Teams.chromium.png|0| Removed|
|Datepicker.Datepicker Header Cell Example.Shows datepicker popup..chromium.png|0| Removed|
|Datepicker.Datepicker Header Cell Example.Shows tooltip on today's date..chromium.png|0| Removed|
|Datepicker.Datepicker Header Cell Example.chromium.png|0| Removed|
|Datepicker.Datepicker Header Example.chromium.png|0| Removed|
|Datepicker.Datepicker No Input Placeholder Example.chromium.png|0| Removed|
|Datepicker.Datepicker Parse Example.chromium.png|0| Removed|

</details>


  <details><summary><b>Dialog</b> 65 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Dialog.DialogExample.chromium.png|0| Added|
|Dialog.DialogExampleBackdrop.chromium.png|0| Added|
|Dialog.DialogExampleBackdropTeams.chromium.png|0| Added|
|Dialog.DialogExampleBackdropTeamsDark.chromium.png|0| Added|
|Dialog.DialogExampleBackdropTeamsHighContrast.chromium.png|0| Added|
|Dialog.DialogExampleCallbacks.chromium.png|0| Added|
|Dialog.DialogExampleContent.chromium.png|0| Added|
|Dialog.DialogExampleContentTeams.chromium.png|0| Added|
|Dialog.DialogExampleContentTeamsDark.chromium.png|0| Added|
|Dialog.DialogExampleContentTeamsHighContrast.chromium.png|0| Added|
|Dialog.DialogExampleFooter.chromium.png|0| Added|
|Dialog.DialogExampleFullWidth.chromium.png|0| Added|
|Dialog.DialogExampleHeaderAction.chromium.png|0| Added|
|Dialog.DialogExampleHeaderActionTeams.chromium.png|0| Added|
|Dialog.DialogExampleHeaderActionTeamsDark.chromium.png|0| Added|
|Dialog.DialogExampleHeaderActionTeamsHighContrast.chromium.png|0| Added|
|Dialog.DialogExampleRtl.chromium.png|0| Added|
|Dialog.DialogExampleScroll.chromium.png|0| Added|
|Dialog.DialogExampleScrollTeams.chromium.png|0| Added|
|Dialog.DialogExampleScrollTeamsDark.chromium.png|0| Added|
|Dialog.DialogExampleScrollTeamsHighContrast.chromium.png|0| Added|
|Dialog.DialogExampleZoomContent.chromium.png|0| Added|
|Dialog.DialogExampleZoomContentTeams.chromium.png|0| Added|
|Dialog.DialogExampleZoomContentTeamsDark.chromium.png|0| Added|
|Dialog.DialogExampleZoomContentTeamsHighContrast.chromium.png|0| Added|
|Dialog.Dialog Example Backdrop Teams Dark.chromium.png|0| Removed|
|Dialog.Dialog Example Backdrop Teams High Contrast.chromium.png|0| Removed|
|Dialog.Dialog Example Backdrop Teams.chromium.png|0| Removed|
|Dialog.Dialog Example Backdrop.chromium.png|0| Removed|
|Dialog.Dialog Example Callbacks.chromium.png|0| Removed|
|Dialog.Dialog Example Content Teams Dark.Clicks the trigger button.chromium.png|0| Removed|
|Dialog.Dialog Example Content Teams Dark.chromium.png|0| Removed|
|Dialog.Dialog Example Content Teams High Contrast.Clicks the trigger button.chromium.png|0| Removed|
|Dialog.Dialog Example Content Teams High Contrast.chromium.png|0| Removed|
|Dialog.Dialog Example Content Teams.Clicks the trigger button.chromium.png|0| Removed|
|Dialog.Dialog Example Content Teams.chromium.png|0| Removed|
|Dialog.Dialog Example Content.Clicks the trigger button.chromium.png|0| Removed|
|Dialog.Dialog Example Content.chromium.png|0| Removed|
|Dialog.Dialog Example Footer.Clicks the trigger button.chromium.png|0| Removed|
|Dialog.Dialog Example Footer.chromium.png|0| Removed|
|Dialog.Dialog Example Full Width.Clicks the trigger button.chromium.png|0| Removed|
|Dialog.Dialog Example Full Width.chromium.png|0| Removed|
|Dialog.Dialog Example Header Action Teams Dark.chromium.png|0| Removed|
|Dialog.Dialog Example Header Action Teams High Contrast.chromium.png|0| Removed|
|Dialog.Dialog Example Header Action Teams.chromium.png|0| Removed|
|Dialog.Dialog Example Header Action.chromium.png|0| Removed|
|Dialog.Dialog Example Rtl.Clicks the trigger button.chromium.png|0| Removed|
|Dialog.Dialog Example Rtl.chromium.png|0| Removed|
|Dialog.Dialog Example Scroll Teams Dark.Clicks the trigger button.chromium.png|0| Removed|
|Dialog.Dialog Example Scroll Teams Dark.chromium.png|0| Removed|
|Dialog.Dialog Example Scroll Teams High Contrast.Clicks the trigger button.chromium.png|0| Removed|
|Dialog.Dialog Example Scroll Teams High Contrast.chromium.png|0| Removed|
|Dialog.Dialog Example Scroll Teams.Clicks the trigger button.chromium.png|0| Removed|
|Dialog.Dialog Example Scroll Teams.chromium.png|0| Removed|
|Dialog.Dialog Example Scroll.Clicks the trigger button.chromium.png|0| Removed|
|Dialog.Dialog Example Scroll.chromium.png|0| Removed|
|Dialog.Dialog Example Zoom Content Teams Dark.Clicks the trigger button.chromium.png|0| Removed|
|Dialog.Dialog Example Zoom Content Teams Dark.chromium.png|0| Removed|
|Dialog.Dialog Example Zoom Content Teams High Contrast.Clicks the trigger button.chromium.png|0| Removed|
|Dialog.Dialog Example Zoom Content Teams High Contrast.chromium.png|0| Removed|
|Dialog.Dialog Example Zoom Content Teams.Clicks the trigger button.chromium.png|0| Removed|
|Dialog.Dialog Example Zoom Content Teams.chromium.png|0| Removed|
|Dialog.Dialog Example Zoom Content.Clicks the trigger button.chromium.png|0| Removed|
|Dialog.Dialog Example Zoom Content.chromium.png|0| Removed|
|Dialog.Dialog Example.chromium.png|0| Removed|

</details>


  <details><summary><b>Divider</b> 68 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Divider.DividerExampleColor.chromium.png|0| Added|
|Divider.DividerExampleColorTeams.chromium.png|0| Added|
|Divider.DividerExampleColorTeamsDark.chromium.png|0| Added|
|Divider.DividerExampleColorTeamsHighContrast.chromium.png|0| Added|
|Divider.DividerExampleContent.chromium.png|0| Added|
|Divider.DividerExampleContentTeams.chromium.png|0| Added|
|Divider.DividerExampleContentTeamsDark.chromium.png|0| Added|
|Divider.DividerExampleContentTeamsHighContrast.chromium.png|0| Added|
|Divider.DividerExampleFittedShorthand.chromium.png|0| Added|
|Divider.DividerExampleFittedShorthandTeams.chromium.png|0| Added|
|Divider.DividerExampleFittedShorthandTeamsDark.chromium.png|0| Added|
|Divider.DividerExampleFittedShorthandTeamsHighContrast.chromium.png|0| Added|
|Divider.DividerExampleImportant.chromium.png|0| Added|
|Divider.DividerExampleImportantShorthand.chromium.png|0| Added|
|Divider.DividerExampleImportantShorthandTeams.chromium.png|0| Added|
|Divider.DividerExampleImportantShorthandTeamsDark.chromium.png|0| Added|
|Divider.DividerExampleImportantShorthandTeamsHighContrast.chromium.png|0| Added|
|Divider.DividerExampleRtl.chromium.png|0| Added|
|Divider.DividerExampleRtlAndLtr.chromium.png|0| Added|
|Divider.DividerExampleRtlAndLtrTeams.chromium.png|0| Added|
|Divider.DividerExampleRtlAndLtrTeamsDark.chromium.png|0| Added|
|Divider.DividerExampleRtlAndLtrTeamsHighContrast.chromium.png|0| Added|
|Divider.DividerExampleRtlTeams.chromium.png|0| Added|
|Divider.DividerExampleRtlTeamsDark.chromium.png|0| Added|
|Divider.DividerExampleRtlTeamsHighContrast.chromium.png|0| Added|
|Divider.DividerExampleShorthand.chromium.png|0| Added|
|Divider.DividerExampleShorthandTeams.chromium.png|0| Added|
|Divider.DividerExampleShorthandTeamsDark.chromium.png|0| Added|
|Divider.DividerExampleShorthandTeamsHighContrast.chromium.png|0| Added|
|Divider.DividerExampleSize.chromium.png|0| Added|
|Divider.DividerExampleSizeTeams.chromium.png|0| Added|
|Divider.DividerExampleSizeTeamsDark.chromium.png|0| Added|
|Divider.DividerExampleSizeTeamsHighContrast.chromium.png|0| Added|
|Divider.DividerVerticalExampleShorthand.chromium.png|0| Added|
|Divider.Divider Example Color Teams Dark.chromium.png|0| Removed|
|Divider.Divider Example Color Teams High Contrast.chromium.png|0| Removed|
|Divider.Divider Example Color Teams.chromium.png|0| Removed|
|Divider.Divider Example Color.chromium.png|0| Removed|
|Divider.Divider Example Content Teams Dark.chromium.png|0| Removed|
|Divider.Divider Example Content Teams High Contrast.chromium.png|0| Removed|
|Divider.Divider Example Content Teams.chromium.png|0| Removed|
|Divider.Divider Example Content.chromium.png|0| Removed|
|Divider.Divider Example Fitted Shorthand Teams Dark.chromium.png|0| Removed|
|Divider.Divider Example Fitted Shorthand Teams High Contrast.chromium.png|0| Removed|
|Divider.Divider Example Fitted Shorthand Teams.chromium.png|0| Removed|
|Divider.Divider Example Fitted Shorthand.chromium.png|0| Removed|
|Divider.Divider Example Important Shorthand Teams Dark.chromium.png|0| Removed|
|Divider.Divider Example Important Shorthand Teams High Contrast.chromium.png|0| Removed|
|Divider.Divider Example Important Shorthand Teams.chromium.png|0| Removed|
|Divider.Divider Example Important Shorthand.chromium.png|0| Removed|
|Divider.Divider Example Important.chromium.png|0| Removed|
|Divider.Divider Example Rtl And Ltr Teams Dark.chromium.png|0| Removed|
|Divider.Divider Example Rtl And Ltr Teams High Contrast.chromium.png|0| Removed|
|Divider.Divider Example Rtl And Ltr Teams.chromium.png|0| Removed|
|Divider.Divider Example Rtl And Ltr.chromium.png|0| Removed|
|Divider.Divider Example Rtl Teams Dark.chromium.png|0| Removed|
|Divider.Divider Example Rtl Teams High Contrast.chromium.png|0| Removed|
|Divider.Divider Example Rtl Teams.chromium.png|0| Removed|
|Divider.Divider Example Rtl.chromium.png|0| Removed|
|Divider.Divider Example Shorthand Teams Dark.chromium.png|0| Removed|
|Divider.Divider Example Shorthand Teams High Contrast.chromium.png|0| Removed|
|Divider.Divider Example Shorthand Teams.chromium.png|0| Removed|
|Divider.Divider Example Shorthand.chromium.png|0| Removed|
|Divider.Divider Example Size Teams Dark.chromium.png|0| Removed|
|Divider.Divider Example Size Teams High Contrast.chromium.png|0| Removed|
|Divider.Divider Example Size Teams.chromium.png|0| Removed|
|Divider.Divider Example Size.chromium.png|0| Removed|
|Divider.Divider Vertical Example Shorthand.chromium.png|0| Removed|

</details>


  <details><summary><b>Dropdown</b> 98 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Dropdown.DropdownClearableExample.chromium.png|0| Added|
|Dropdown.DropdownCustomClearableExample.chromium.png|0| Added|
|Dropdown.DropdownCustomClearableExampleTeams.chromium.png|0| Added|
|Dropdown.DropdownCustomClearableExampleTeamsDark.chromium.png|0| Added|
|Dropdown.DropdownCustomClearableExampleTeamsHighContrast.chromium.png|0| Added|
|Dropdown.DropdownExample.chromium.png|0| Added|
|Dropdown.DropdownExampleDisabled.chromium.png|0| Added|
|Dropdown.DropdownExampleDisabledTeams.chromium.png|0| Added|
|Dropdown.DropdownExampleDisabledTeamsDark.chromium.png|0| Added|
|Dropdown.DropdownExampleDisabledTeamsHighContrast.chromium.png|0| Added|
|Dropdown.DropdownExampleFreeform.chromium.png|0| Added|
|Dropdown.DropdownExampleHeaderMessage.chromium.png|0| Added|
|Dropdown.DropdownExampleInline.chromium.png|0| Added|
|Dropdown.DropdownExampleInverted.chromium.png|0| Added|
|Dropdown.DropdownExampleLoading.chromium.png|0| Added|
|Dropdown.DropdownExampleLoadingTeams.chromium.png|0| Added|
|Dropdown.DropdownExampleLoadingTeamsDark.chromium.png|0| Added|
|Dropdown.DropdownExampleLoadingTeamsHighContrast.chromium.png|0| Added|
|Dropdown.DropdownExampleMultiple.chromium.png|0| Added|
|Dropdown.DropdownExamplePosition.chromium.png|0| Added|
|Dropdown.DropdownExampleRender.chromium.png|0| Added|
|Dropdown.DropdownExampleRtl.chromium.png|0| Added|
|Dropdown.DropdownExampleSearch.chromium.png|0| Added|
|Dropdown.DropdownExampleSearchMultiple.chromium.png|0| Added|
|Dropdown.DropdownExampleSearchMultipleFluid.chromium.png|0| Added|
|Dropdown.DropdownExampleSearchMultipleFrenchLanguage.chromium.png|0| Added|
|Dropdown.DropdownExampleSearchMultipleImageAndContent.chromium.png|0| Added|
|Dropdown.DropdownExampleSearchMultipleTeams.chromium.png|0| Added|
|Dropdown.DropdownExampleSearchMultipleTeamsDark.chromium.png|0| Added|
|Dropdown.DropdownExampleSearchMultipleTeamsHighContrast.chromium.png|0| Added|
|Dropdown.DropdownExampleTeams.chromium.png|0| Added|
|Dropdown.DropdownExampleTeamsDark.chromium.png|0| Added|
|Dropdown.DropdownExampleTeamsHighContrast.chromium.png|0| Added|
|Dropdown.Dropdown Clearable Example.Clears the value.chromium.png|0| Removed|
|Dropdown.Dropdown Clearable Example.Selects an item.chromium.png|0| Removed|
|Dropdown.Dropdown Clearable Example.chromium.png|0| Removed|
|Dropdown.Dropdown Custom Clearable Example Teams Dark.chromium.png|0| Removed|
|Dropdown.Dropdown Custom Clearable Example Teams High Contrast.chromium.png|0| Removed|
|Dropdown.Dropdown Custom Clearable Example Teams.chromium.png|0| Removed|
|Dropdown.Dropdown Custom Clearable Example.chromium.png|0| Removed|
|Dropdown.Dropdown Example Disabled Teams Dark.Mouse hover on input.chromium.png|0| Removed|
|Dropdown.Dropdown Example Disabled Teams Dark.chromium.png|0| Removed|
|Dropdown.Dropdown Example Disabled Teams High Contrast.Mouse hover on input.chromium.png|0| Removed|
|Dropdown.Dropdown Example Disabled Teams High Contrast.chromium.png|0| Removed|
|Dropdown.Dropdown Example Disabled Teams.Mouse hover on input.chromium.png|0| Removed|
|Dropdown.Dropdown Example Disabled Teams.chromium.png|0| Removed|
|Dropdown.Dropdown Example Disabled.Mouse hover on input.chromium.png|0| Removed|
|Dropdown.Dropdown Example Disabled.chromium.png|0| Removed|
|Dropdown.Dropdown Example Freeform.chromium.png|0| Removed|
|Dropdown.Dropdown Example Header Message.Shows header message.chromium.png|0| Removed|
|Dropdown.Dropdown Example Header Message.chromium.png|0| Removed|
|Dropdown.Dropdown Example Inline.chromium.png|0| Removed|
|Dropdown.Dropdown Example Inverted.chromium.png|0| Removed|
|Dropdown.Dropdown Example Loading Teams Dark.chromium.png|0| Removed|
|Dropdown.Dropdown Example Loading Teams High Contrast.chromium.png|0| Removed|
|Dropdown.Dropdown Example Loading Teams.chromium.png|0| Removed|
|Dropdown.Dropdown Example Loading.chromium.png|0| Removed|
|Dropdown.Dropdown Example Multiple.Opened dropdown with no items selected.chromium.png|0| Removed|
|Dropdown.Dropdown Example Multiple.Opened dropdown with two items selected.chromium.png|0| Removed|
|Dropdown.Dropdown Example Multiple.chromium.png|0| Removed|
|Dropdown.Dropdown Example Position.Opens a dropdown.chromium.png|0| Removed|
|Dropdown.Dropdown Example Position.chromium.png|0| Removed|
|Dropdown.Dropdown Example Render.chromium.png|0| Removed|
|Dropdown.Dropdown Example Rtl.RTL- Selects an item.chromium.png|0| Removed|
|Dropdown.Dropdown Example Rtl.RTL- Shows list.chromium.png|0| Removed|
|Dropdown.Dropdown Example Rtl.chromium.png|0| Removed|
|Dropdown.Dropdown Example Search Multiple Fluid.chromium.png|0| Removed|
|Dropdown.Dropdown Example Search Multiple French Language.chromium.png|0| Removed|
|Dropdown.Dropdown Example Search Multiple Image And Content.chromium.png|0| Removed|
|Dropdown.Dropdown Example Search Multiple Teams Dark.Hovers first selected element.chromium.png|0| Removed|
|Dropdown.Dropdown Example Search Multiple Teams Dark.Selects last selected element.chromium.png|0| Removed|
|Dropdown.Dropdown Example Search Multiple Teams Dark.chromium.png|0| Removed|
|Dropdown.Dropdown Example Search Multiple Teams High Contrast.Hovers first selected element.chromium.png|0| Removed|
|Dropdown.Dropdown Example Search Multiple Teams High Contrast.Selects last selected element.chromium.png|0| Removed|
|Dropdown.Dropdown Example Search Multiple Teams High Contrast.chromium.png|0| Removed|
|Dropdown.Dropdown Example Search Multiple Teams.Hovers first selected element.chromium.png|0| Removed|
|Dropdown.Dropdown Example Search Multiple Teams.Selects last selected element.chromium.png|0| Removed|
|Dropdown.Dropdown Example Search Multiple Teams.chromium.png|0| Removed|
|Dropdown.Dropdown Example Search Multiple.Hovers first selected element.chromium.png|0| Removed|
|Dropdown.Dropdown Example Search Multiple.Selects last selected element.chromium.png|0| Removed|
|Dropdown.Dropdown Example Search Multiple.chromium.png|0| Removed|
|Dropdown.Dropdown Example Search.chromium.png|0| Removed|
|Dropdown.Dropdown Example Teams Dark.Focuses first item.chromium.png|0| Removed|
|Dropdown.Dropdown Example Teams Dark.Focuses second item.chromium.png|0| Removed|
|Dropdown.Dropdown Example Teams Dark.Focuses trigger.chromium.png|0| Removed|
|Dropdown.Dropdown Example Teams Dark.chromium.png|0| Removed|
|Dropdown.Dropdown Example Teams High Contrast.Focuses first item.chromium.png|0| Removed|
|Dropdown.Dropdown Example Teams High Contrast.Focuses second item.chromium.png|0| Removed|
|Dropdown.Dropdown Example Teams High Contrast.Focuses trigger.chromium.png|0| Removed|
|Dropdown.Dropdown Example Teams High Contrast.chromium.png|0| Removed|
|Dropdown.Dropdown Example Teams.Focuses first item.chromium.png|0| Removed|
|Dropdown.Dropdown Example Teams.Focuses second item.chromium.png|0| Removed|
|Dropdown.Dropdown Example Teams.Focuses trigger.chromium.png|0| Removed|
|Dropdown.Dropdown Example Teams.chromium.png|0| Removed|
|Dropdown.Dropdown Example.Focuses first item.chromium.png|0| Removed|
|Dropdown.Dropdown Example.Focuses second item.chromium.png|0| Removed|
|Dropdown.Dropdown Example.Focuses trigger.chromium.png|0| Removed|
|Dropdown.Dropdown Example.chromium.png|0| Removed|

</details>


  <details><summary><b>Embed</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Embed.EmbedExampleVideo.chromium.png|0| Added|
|Embed.EmbedExampleYouTube.chromium.png|0| Added|
|Embed.Embed Example Video.chromium.png|0| Removed|
|Embed.Embed Example You Tube.chromium.png|0| Removed|

</details>


  <details><summary><b>Flex</b> 12 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Flex.FlexExampleColumns.chromium.png|0| Added|
|Flex.FlexExampleInput.chromium.png|0| Added|
|Flex.FlexExampleItemsAlignment.chromium.png|0| Added|
|Flex.FlexExampleMediaCard.chromium.png|0| Added|
|Flex.FlexExampleMixedAlignment.chromium.png|0| Added|
|Flex.FlexExampleNavMenu.chromium.png|0| Added|
|Flex.Flex Example Columns.chromium.png|0| Removed|
|Flex.Flex Example Input.chromium.png|0| Removed|
|Flex.Flex Example Items Alignment.chromium.png|0| Removed|
|Flex.Flex Example Media Card.chromium.png|0| Removed|
|Flex.Flex Example Mixed Alignment.chromium.png|0| Removed|
|Flex.Flex Example Nav Menu.chromium.png|0| Removed|

</details>


  <details><summary><b>Form</b> 14 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Form.FormExample.chromium.png|0| Added|
|Form.FormExampleCheckbox.chromium.png|0| Added|
|Form.FormExampleComponents.chromium.png|0| Added|
|Form.FormExampleDatepicker.chromium.png|0| Added|
|Form.FormExampleErrorAndSatisfactory.chromium.png|0| Added|
|Form.FormExampleRtl.chromium.png|0| Added|
|Form.FormExampleSlider.chromium.png|0| Added|
|Form.Form Example Checkbox.chromium.png|0| Removed|
|Form.Form Example Components.chromium.png|0| Removed|
|Form.Form Example Datepicker.chromium.png|0| Removed|
|Form.Form Example Error And Satisfactory.chromium.png|0| Removed|
|Form.Form Example Rtl.chromium.png|0| Removed|
|Form.Form Example Slider.chromium.png|0| Removed|
|Form.Form Example.chromium.png|0| Removed|

</details>


  <details><summary><b>Grid</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Grid.GridExample.chromium.png|0| Added|
|Grid.Grid Example.chromium.png|0| Removed|

</details>


  <details><summary><b>Header</b> 18 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Header.HeaderExample.chromium.png|0| Added|
|Header.HeaderExampleAlign.chromium.png|0| Added|
|Header.HeaderExampleColor.chromium.png|0| Added|
|Header.HeaderExampleDescription.chromium.png|0| Added|
|Header.HeaderExampleDescriptionCustomization.chromium.png|0| Added|
|Header.HeaderExampleDescriptionCustomizationShorthand.chromium.png|0| Added|
|Header.HeaderExampleDescriptionShorthand.chromium.png|0| Added|
|Header.HeaderExampleRtl.chromium.png|0| Added|
|Header.HeadersExampleShorthand.chromium.png|0| Added|
|Header.Header Example Align.chromium.png|0| Removed|
|Header.Header Example Color.chromium.png|0| Removed|
|Header.Header Example Description Customization Shorthand.chromium.png|0| Removed|
|Header.Header Example Description Customization.chromium.png|0| Removed|
|Header.Header Example Description Shorthand.chromium.png|0| Removed|
|Header.Header Example Description.chromium.png|0| Removed|
|Header.Header Example Rtl.chromium.png|0| Removed|
|Header.Header Example.chromium.png|0| Removed|
|Header.Headers Example Shorthand.chromium.png|0| Removed|

</details>


  <details><summary><b>Image</b> 10 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Image.ImageExample.chromium.png|0| Added|
|Image.ImageExampleAvatar.chromium.png|0| Added|
|Image.ImageExampleCircular.chromium.png|0| Added|
|Image.ImageExampleFluent.chromium.png|0| Added|
|Image.ImageExampleRtl.chromium.png|0| Added|
|Image.Image Example Avatar.chromium.png|0| Removed|
|Image.Image Example Circular.chromium.png|0| Removed|
|Image.Image Example Fluent.chromium.png|0| Removed|
|Image.Image Example Rtl.chromium.png|0| Removed|
|Image.Image Example.chromium.png|0| Removed|

</details>


  <details><summary><b>Input</b> 52 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Input.InputExample.chromium.png|0| Added|
|Input.InputExampleClearable.chromium.png|0| Added|
|Input.InputExampleClearableTeams.chromium.png|0| Added|
|Input.InputExampleClearableTeamsDark.chromium.png|0| Added|
|Input.InputExampleClearableTeamsHighContrast.chromium.png|0| Added|
|Input.InputExampleDisabled.chromium.png|0| Added|
|Input.InputExampleFluid.chromium.png|0| Added|
|Input.InputExampleIcon.chromium.png|0| Added|
|Input.InputExampleIconClearableShorthand.chromium.png|0| Added|
|Input.InputExampleIconPosition.chromium.png|0| Added|
|Input.InputExampleInline.chromium.png|0| Added|
|Input.InputExampleInputSlot.chromium.png|0| Added|
|Input.InputExampleLabelShorthand.chromium.png|0| Added|
|Input.InputExampleLabelShorthandTeams.chromium.png|0| Added|
|Input.InputExampleLabelShorthandTeamsDark.chromium.png|0| Added|
|Input.InputExampleLabelShorthandTeamsHighContrast.chromium.png|0| Added|
|Input.InputExampleRtl.chromium.png|0| Added|
|Input.InputExampleTargeting.chromium.png|0| Added|
|Input.InputExampleTeams.chromium.png|0| Added|
|Input.InputExampleTeamsDark.chromium.png|0| Added|
|Input.InputExampleTeamsHighContrast.chromium.png|0| Added|
|Input.InputExampleWrapperSlot.chromium.png|0| Added|
|Input.Input Example Clearable Teams Dark.Can be clearable.chromium.png|0| Removed|
|Input.Input Example Clearable Teams Dark.chromium.png|0| Removed|
|Input.Input Example Clearable Teams High Contrast.Can be clearable.chromium.png|0| Removed|
|Input.Input Example Clearable Teams High Contrast.chromium.png|0| Removed|
|Input.Input Example Clearable Teams.Can be clearable.chromium.png|0| Removed|
|Input.Input Example Clearable Teams.chromium.png|0| Removed|
|Input.Input Example Clearable.Can be clearable.chromium.png|0| Removed|
|Input.Input Example Clearable.chromium.png|0| Removed|
|Input.Input Example Disabled.chromium.png|0| Removed|
|Input.Input Example Fluid.chromium.png|0| Removed|
|Input.Input Example Icon Clearable Shorthand.chromium.png|0| Removed|
|Input.Input Example Icon Position.chromium.png|0| Removed|
|Input.Input Example Icon.chromium.png|0| Removed|
|Input.Input Example Inline.chromium.png|0| Removed|
|Input.Input Example Input Slot.chromium.png|0| Removed|
|Input.Input Example Label Shorthand Teams Dark.Input- Inside Label with Value.chromium.png|0| Removed|
|Input.Input Example Label Shorthand Teams Dark.Input- Labels.chromium.png|0| Removed|
|Input.Input Example Label Shorthand Teams High Contrast.Input- Inside Label with Value.chromium.png|0| Removed|
|Input.Input Example Label Shorthand Teams High Contrast.Input- Labels.chromium.png|0| Removed|
|Input.Input Example Label Shorthand Teams.Input- Inside Label with Value.chromium.png|0| Removed|
|Input.Input Example Label Shorthand Teams.Input- Labels.chromium.png|0| Removed|
|Input.Input Example Label Shorthand.Input- Inside Label with Value.chromium.png|0| Removed|
|Input.Input Example Label Shorthand.Input- Labels.chromium.png|0| Removed|
|Input.Input Example Rtl.chromium.png|0| Removed|
|Input.Input Example Targeting.chromium.png|0| Removed|
|Input.Input Example Teams Dark.chromium.png|0| Removed|
|Input.Input Example Teams High Contrast.chromium.png|0| Removed|
|Input.Input Example Teams.chromium.png|0| Removed|
|Input.Input Example Wrapper Slot.chromium.png|0| Removed|
|Input.Input Example.chromium.png|0| Removed|

</details>


  <details><summary><b>ItemLayout</b> 20 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ItemLayout.ItemLayoutExampleContent.chromium.png|0| Added|
|ItemLayout.ItemLayoutExampleContentMediaShorthand.chromium.png|0| Added|
|ItemLayout.ItemLayoutExampleEndMediaShorthand.chromium.png|0| Added|
|ItemLayout.ItemLayoutExampleHeader.chromium.png|0| Added|
|ItemLayout.ItemLayoutExampleHeaderContentShorthand.chromium.png|0| Added|
|ItemLayout.ItemLayoutExampleHeaderMediaShorthand.chromium.png|0| Added|
|ItemLayout.ItemLayoutExampleMediaShorthand.chromium.png|0| Added|
|ItemLayout.ItemLayoutExampleRtlShorthand.chromium.png|0| Added|
|ItemLayout.ItemLayoutExampleSelectionShorthand.chromium.png|0| Added|
|ItemLayout.ItemLayoutExampleShorthand.chromium.png|0| Added|
|ItemLayout.Item Layout Example Content Media Shorthand.chromium.png|0| Removed|
|ItemLayout.Item Layout Example Content.chromium.png|0| Removed|
|ItemLayout.Item Layout Example End Media Shorthand.chromium.png|0| Removed|
|ItemLayout.Item Layout Example Header Content Shorthand.chromium.png|0| Removed|
|ItemLayout.Item Layout Example Header Media Shorthand.chromium.png|0| Removed|
|ItemLayout.Item Layout Example Header.chromium.png|0| Removed|
|ItemLayout.Item Layout Example Media Shorthand.chromium.png|0| Removed|
|ItemLayout.Item Layout Example Rtl Shorthand.chromium.png|0| Removed|
|ItemLayout.Item Layout Example Selection Shorthand.chromium.png|0| Removed|
|ItemLayout.Item Layout Example Shorthand.chromium.png|0| Removed|

</details>


  <details><summary><b>Label</b> 18 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Label.LabelExample.chromium.png|0| Added|
|Label.LabelExampleCircularShorthand.chromium.png|0| Added|
|Label.LabelExampleColor.chromium.png|0| Added|
|Label.LabelExampleContentCustomizationShorthand.chromium.png|0| Added|
|Label.LabelExampleIconPositionShorthand.chromium.png|0| Added|
|Label.LabelExampleIconShorthand.chromium.png|0| Added|
|Label.LabelExampleImageShorthand.chromium.png|0| Added|
|Label.LabelExampleRtl.chromium.png|0| Added|
|Label.LabelExampleShorthand.chromium.png|0| Added|
|Label.Label Example Circular Shorthand.chromium.png|0| Removed|
|Label.Label Example Color.chromium.png|0| Removed|
|Label.Label Example Content Customization Shorthand.chromium.png|0| Removed|
|Label.Label Example Icon Position Shorthand.chromium.png|0| Removed|
|Label.Label Example Icon Shorthand.chromium.png|0| Removed|
|Label.Label Example Image Shorthand.chromium.png|0| Removed|
|Label.Label Example Rtl.chromium.png|0| Removed|
|Label.Label Example Shorthand.chromium.png|0| Removed|
|Label.Label Example.chromium.png|0| Removed|

</details>


  <details><summary><b>Layout</b> 10 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Layout.LayoutExample.chromium.png|0| Added|
|Layout.LayoutExampleDisappearing.chromium.png|0| Added|
|Layout.LayoutExampleGap.chromium.png|0| Added|
|Layout.LayoutExampleReducing.chromium.png|0| Added|
|Layout.LayoutExampleVertical.chromium.png|0| Added|
|Layout.Layout Example Disappearing.chromium.png|0| Removed|
|Layout.Layout Example Gap.chromium.png|0| Removed|
|Layout.Layout Example Reducing.chromium.png|0| Removed|
|Layout.Layout Example Vertical.chromium.png|0| Removed|
|Layout.Layout Example.chromium.png|0| Removed|

</details>


  <details><summary><b>List</b> 34 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|List.ListExample.chromium.png|0| Added|
|List.ListExampleMedia.chromium.png|0| Added|
|List.ListExampleMediaShorthand.chromium.png|0| Added|
|List.ListExampleNavigable.chromium.png|0| Added|
|List.ListExampleNavigableTeams.chromium.png|0| Added|
|List.ListExampleNavigableTeamsDark.chromium.png|0| Added|
|List.ListExampleNavigableTeamsHighContrast.chromium.png|0| Added|
|List.ListExampleRtl.chromium.png|0| Added|
|List.ListExampleSelectable.chromium.png|0| Added|
|List.ListExampleSelectableTeams.chromium.png|0| Added|
|List.ListExampleSelectableTeamsDark.chromium.png|0| Added|
|List.ListExampleSelectableTeamsHighContrast.chromium.png|0| Added|
|List.SelectableListControlledExample.chromium.png|0| Added|
|List.List Example Media Shorthand.chromium.png|0| Removed|
|List.List Example Media.chromium.png|0| Removed|
|List.List Example Navigable Teams Dark.Focuses item.chromium.png|0| Removed|
|List.List Example Navigable Teams Dark.chromium.png|0| Removed|
|List.List Example Navigable Teams High Contrast.Focuses item.chromium.png|0| Removed|
|List.List Example Navigable Teams High Contrast.chromium.png|0| Removed|
|List.List Example Navigable Teams.Focuses item.chromium.png|0| Removed|
|List.List Example Navigable Teams.chromium.png|0| Removed|
|List.List Example Navigable.Focuses item.chromium.png|0| Removed|
|List.List Example Navigable.chromium.png|0| Removed|
|List.List Example Rtl.chromium.png|0| Removed|
|List.List Example Selectable Teams Dark.Focuses item.chromium.png|0| Removed|
|List.List Example Selectable Teams Dark.chromium.png|0| Removed|
|List.List Example Selectable Teams High Contrast.Focuses item.chromium.png|0| Removed|
|List.List Example Selectable Teams High Contrast.chromium.png|0| Removed|
|List.List Example Selectable Teams.Focuses item.chromium.png|0| Removed|
|List.List Example Selectable Teams.chromium.png|0| Removed|
|List.List Example Selectable.Focuses item.chromium.png|0| Removed|
|List.List Example Selectable.chromium.png|0| Removed|
|List.List Example.chromium.png|0| Removed|
|List.Selectable List Controlled Example.chromium.png|0| Removed|

</details>


  <details><summary><b>Loader</b> 14 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Loader.LoaderExample.chromium.png|0| Added|
|Loader.LoaderExampleDelay.chromium.png|0| Added|
|Loader.LoaderExampleInline.chromium.png|0| Added|
|Loader.LoaderExampleLabel.chromium.png|0| Added|
|Loader.LoaderExampleRtl.chromium.png|0| Added|
|Loader.LoaderExampleSecondary.chromium.png|0| Added|
|Loader.LoaderExampleSize.chromium.png|0| Added|
|Loader.Loader Example Delay.chromium.png|0| Removed|
|Loader.Loader Example Inline.chromium.png|0| Removed|
|Loader.Loader Example Label.chromium.png|0| Removed|
|Loader.Loader Example Rtl.chromium.png|0| Removed|
|Loader.Loader Example Secondary.chromium.png|0| Removed|
|Loader.Loader Example Size.chromium.png|0| Removed|
|Loader.Loader Example.chromium.png|0| Removed|

</details>


  <details><summary><b>Menu</b> 220 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu.MenuControlledExampleDisabled.chromium.png|0| Added|
|Menu.MenuExample.chromium.png|0| Added|
|Menu.MenuExampleDividerHorizontal.chromium.png|0| Added|
|Menu.MenuExampleFluid.chromium.png|0| Added|
|Menu.MenuExampleIconOnly.chromium.png|0| Added|
|Menu.MenuExampleIconOnlyTeams.chromium.png|0| Added|
|Menu.MenuExampleIconOnlyTeamsDark.chromium.png|0| Added|
|Menu.MenuExampleIconOnlyTeamsHighContrast.chromium.png|0| Added|
|Menu.MenuExampleKind.chromium.png|0| Added|
|Menu.MenuExamplePointing.chromium.png|0| Added|
|Menu.MenuExamplePointingTeams.chromium.png|0| Added|
|Menu.MenuExamplePointingTeamsDark.chromium.png|0| Added|
|Menu.MenuExamplePointingTeamsHighContrast.chromium.png|0| Added|
|Menu.MenuExamplePositioningShorthand.chromium.png|0| Added|
|Menu.MenuExamplePositioningUpdateShorthand.chromium.png|0| Added|
|Menu.MenuExampleSlot.chromium.png|0| Added|
|Menu.MenuExampleTabShorthand.chromium.png|0| Added|
|Menu.MenuExampleTeams.chromium.png|0| Added|
|Menu.MenuExampleTeamsDark.chromium.png|0| Added|
|Menu.MenuExampleTeamsHighContrast.chromium.png|0| Added|
|Menu.MenuExampleToolbarShorthand.chromium.png|0| Added|
|Menu.MenuExampleToolbarShorthandTeams.chromium.png|0| Added|
|Menu.MenuExampleToolbarShorthandTeamsDark.chromium.png|0| Added|
|Menu.MenuExampleToolbarShorthandTeamsHighContrast.chromium.png|0| Added|
|Menu.MenuExampleUnderlined.chromium.png|0| Added|
|Menu.MenuExampleUnderlinedTeams.chromium.png|0| Added|
|Menu.MenuExampleUnderlinedTeamsDark.chromium.png|0| Added|
|Menu.MenuExampleUnderlinedTeamsHighContrast.chromium.png|0| Added|
|Menu.MenuExampleVertical.chromium.png|0| Added|
|Menu.MenuExampleVerticalPointing.chromium.png|0| Added|
|Menu.MenuExampleVerticalTeams.chromium.png|0| Added|
|Menu.MenuExampleVerticalTeamsDark.chromium.png|0| Added|
|Menu.MenuExampleVerticalTeamsHighContrast.chromium.png|0| Added|
|Menu.MenuExampleWithIcons.chromium.png|0| Added|
|Menu.MenuExampleWithSubMenu.chromium.png|0| Added|
|Menu.MenuExampleWithSubMenuHover.chromium.png|0| Added|
|Menu.MenuExampleWithSubMenuHoverTeams.chromium.png|0| Added|
|Menu.MenuExampleWithSubMenuHoverTeamsDark.chromium.png|0| Added|
|Menu.MenuExampleWithSubMenuHoverTeamsHighContrast.chromium.png|0| Added|
|Menu.MenuExampleWithTooltip.chromium.png|0| Added|
|Menu.MenuExampleWithTooltipTeams.chromium.png|0| Added|
|Menu.MenuExampleWithTooltipTeamsDark.chromium.png|0| Added|
|Menu.MenuExampleWithTooltipTeamsHighContrast.chromium.png|0| Added|
|Menu.MenuItemExampleDisabled.chromium.png|0| Added|
|Menu.MenuPlayground.chromium.png|0| Added|
|Menu.MenuWithSubmenuControlledExample.chromium.png|0| Added|
|Menu.Menu Controlled Example Disabled.chromium.png|0| Removed|
|Menu.Menu Example Divider Horizontal.chromium.png|0| Removed|
|Menu.Menu Example Fluid.chromium.png|0| Removed|
|Menu.Menu Example Icon Only Teams Dark.Clicks on 2nd item (active state styles).chromium.png|0| Removed|
|Menu.Menu Example Icon Only Teams Dark.Hovers 2nd item (hover state styles).chromium.png|0| Removed|
|Menu.Menu Example Icon Only Teams Dark.Navigates to next item (focus state styles).chromium.png|0| Removed|
|Menu.Menu Example Icon Only Teams Dark.Navigates to previous item (active and focus state styles).chromium.png|0| Removed|
|Menu.Menu Example Icon Only Teams Dark.chromium.png|0| Removed|
|Menu.Menu Example Icon Only Teams High Contrast.Clicks on 2nd item (active state styles).chromium.png|0| Removed|
|Menu.Menu Example Icon Only Teams High Contrast.Hovers 2nd item (hover state styles).chromium.png|0| Removed|
|Menu.Menu Example Icon Only Teams High Contrast.Navigates to next item (focus state styles).chromium.png|0| Removed|
|Menu.Menu Example Icon Only Teams High Contrast.Navigates to previous item (active and focus state styles).chromium.png|0| Removed|
|Menu.Menu Example Icon Only Teams High Contrast.chromium.png|0| Removed|
|Menu.Menu Example Icon Only Teams.Clicks on 2nd item (active state styles).chromium.png|0| Removed|
|Menu.Menu Example Icon Only Teams.Hovers 2nd item (hover state styles).chromium.png|0| Removed|
|Menu.Menu Example Icon Only Teams.Navigates to next item (focus state styles).chromium.png|0| Removed|
|Menu.Menu Example Icon Only Teams.Navigates to previous item (active and focus state styles).chromium.png|0| Removed|
|Menu.Menu Example Icon Only Teams.chromium.png|0| Removed|
|Menu.Menu Example Icon Only.Clicks on 2nd item (active state styles).chromium.png|0| Removed|
|Menu.Menu Example Icon Only.Hovers 2nd item (hover state styles).chromium.png|0| Removed|
|Menu.Menu Example Icon Only.Navigates to next item (focus state styles).chromium.png|0| Removed|
|Menu.Menu Example Icon Only.Navigates to previous item (active and focus state styles).chromium.png|0| Removed|
|Menu.Menu Example Icon Only.chromium.png|0| Removed|
|Menu.Menu Example Kind.chromium.png|0| Removed|
|Menu.Menu Example Pointing Teams Dark.Clicks on 2nd item (active state styles).chromium.png|0| Removed|
|Menu.Menu Example Pointing Teams Dark.Hovers 2nd item (hover state styles).chromium.png|0| Removed|
|Menu.Menu Example Pointing Teams Dark.Navigates to next item (focus state styles).chromium.png|0| Removed|
|Menu.Menu Example Pointing Teams Dark.Navigates to previous item (active and focus state styles).chromium.png|0| Removed|
|Menu.Menu Example Pointing Teams Dark.chromium.png|0| Removed|
|Menu.Menu Example Pointing Teams High Contrast.Clicks on 2nd item (active state styles).chromium.png|0| Removed|
|Menu.Menu Example Pointing Teams High Contrast.Hovers 2nd item (hover state styles).chromium.png|0| Removed|
|Menu.Menu Example Pointing Teams High Contrast.Navigates to next item (focus state styles).chromium.png|0| Removed|
|Menu.Menu Example Pointing Teams High Contrast.Navigates to previous item (active and focus state styles).chromium.png|0| Removed|
|Menu.Menu Example Pointing Teams High Contrast.chromium.png|0| Removed|
|Menu.Menu Example Pointing Teams.Clicks on 2nd item (active state styles).chromium.png|0| Removed|
|Menu.Menu Example Pointing Teams.Hovers 2nd item (hover state styles).chromium.png|0| Removed|
|Menu.Menu Example Pointing Teams.Navigates to next item (focus state styles).chromium.png|0| Removed|
|Menu.Menu Example Pointing Teams.Navigates to previous item (active and focus state styles).chromium.png|0| Removed|
|Menu.Menu Example Pointing Teams.chromium.png|0| Removed|
|Menu.Menu Example Pointing.Clicks on 2nd item (active state styles).chromium.png|0| Removed|
|Menu.Menu Example Pointing.Hovers 2nd item (hover state styles).chromium.png|0| Removed|
|Menu.Menu Example Pointing.Navigates to next item (focus state styles).chromium.png|0| Removed|
|Menu.Menu Example Pointing.Navigates to previous item (active and focus state styles).chromium.png|0| Removed|
|Menu.Menu Example Pointing.chromium.png|0| Removed|
|Menu.Menu Example Positioning Shorthand.chromium.png|0| Removed|
|Menu.Menu Example Positioning Update Shorthand.chromium.png|0| Removed|
|Menu.Menu Example Slot.chromium.png|0| Removed|
|Menu.Menu Example Tab Shorthand.chromium.png|0| Removed|
|Menu.Menu Example Teams Dark.Clicks on 2nd item (active state styles).chromium.png|0| Removed|
|Menu.Menu Example Teams Dark.Hovers 2nd item (hover state styles).chromium.png|0| Removed|
|Menu.Menu Example Teams Dark.Navigates to next item (focus state styles).chromium.png|0| Removed|
|Menu.Menu Example Teams Dark.Navigates to previous item (active and focus state styles).chromium.png|0| Removed|
|Menu.Menu Example Teams Dark.chromium.png|0| Removed|
|Menu.Menu Example Teams High Contrast.Clicks on 2nd item (active state styles).chromium.png|0| Removed|
|Menu.Menu Example Teams High Contrast.Hovers 2nd item (hover state styles).chromium.png|0| Removed|
|Menu.Menu Example Teams High Contrast.Navigates to next item (focus state styles).chromium.png|0| Removed|
|Menu.Menu Example Teams High Contrast.Navigates to previous item (active and focus state styles).chromium.png|0| Removed|
|Menu.Menu Example Teams High Contrast.chromium.png|0| Removed|
|Menu.Menu Example Teams.Clicks on 2nd item (active state styles).chromium.png|0| Removed|
|Menu.Menu Example Teams.Hovers 2nd item (hover state styles).chromium.png|0| Removed|
|Menu.Menu Example Teams.Navigates to next item (focus state styles).chromium.png|0| Removed|
|Menu.Menu Example Teams.Navigates to previous item (active and focus state styles).chromium.png|0| Removed|
|Menu.Menu Example Teams.chromium.png|0| Removed|
|Menu.Menu Example Toolbar Shorthand Teams Dark.Clicks on 2nd item (active state styles).chromium.png|0| Removed|
|Menu.Menu Example Toolbar Shorthand Teams Dark.Clicks on the last item and opens submenu.chromium.png|0| Removed|
|Menu.Menu Example Toolbar Shorthand Teams Dark.Focuses on the first element in the submenu.chromium.png|0| Removed|
|Menu.Menu Example Toolbar Shorthand Teams Dark.Hovers 2nd item (hover state styles).chromium.png|0| Removed|
|Menu.Menu Example Toolbar Shorthand Teams Dark.Navigates to next item (focus state styles).chromium.png|0| Removed|
|Menu.Menu Example Toolbar Shorthand Teams Dark.Navigates to previous item (active and focus state styles).chromium.png|0| Removed|
|Menu.Menu Example Toolbar Shorthand Teams Dark.chromium.png|0| Removed|
|Menu.Menu Example Toolbar Shorthand Teams High Contrast.Clicks on 2nd item (active state styles).chromium.png|0| Removed|
|Menu.Menu Example Toolbar Shorthand Teams High Contrast.Clicks on the last item and opens submenu.chromium.png|0| Removed|
|Menu.Menu Example Toolbar Shorthand Teams High Contrast.Focuses on the first element in the submenu.chromium.png|0| Removed|
|Menu.Menu Example Toolbar Shorthand Teams High Contrast.Hovers 2nd item (hover state styles).chromium.png|0| Removed|
|Menu.Menu Example Toolbar Shorthand Teams High Contrast.Navigates to next item (focus state styles).chromium.png|0| Removed|
|Menu.Menu Example Toolbar Shorthand Teams High Contrast.Navigates to previous item (active and focus state styles).chromium.png|0| Removed|
|Menu.Menu Example Toolbar Shorthand Teams High Contrast.chromium.png|0| Removed|
|Menu.Menu Example Toolbar Shorthand Teams.Clicks on 2nd item (active state styles).chromium.png|0| Removed|
|Menu.Menu Example Toolbar Shorthand Teams.Clicks on the last item and opens submenu.chromium.png|0| Removed|
|Menu.Menu Example Toolbar Shorthand Teams.Focuses on the first element in the submenu.chromium.png|0| Removed|
|Menu.Menu Example Toolbar Shorthand Teams.Hovers 2nd item (hover state styles).chromium.png|0| Removed|
|Menu.Menu Example Toolbar Shorthand Teams.Navigates to next item (focus state styles).chromium.png|0| Removed|
|Menu.Menu Example Toolbar Shorthand Teams.Navigates to previous item (active and focus state styles).chromium.png|0| Removed|
|Menu.Menu Example Toolbar Shorthand Teams.chromium.png|0| Removed|
|Menu.Menu Example Toolbar Shorthand.Clicks on 2nd item (active state styles).chromium.png|0| Removed|
|Menu.Menu Example Toolbar Shorthand.Clicks on the last item and opens submenu.chromium.png|0| Removed|
|Menu.Menu Example Toolbar Shorthand.Focuses on the first element in the submenu.chromium.png|0| Removed|
|Menu.Menu Example Toolbar Shorthand.Hovers 2nd item (hover state styles).chromium.png|0| Removed|
|Menu.Menu Example Toolbar Shorthand.Navigates to next item (focus state styles).chromium.png|0| Removed|
|Menu.Menu Example Toolbar Shorthand.Navigates to previous item (active and focus state styles).chromium.png|0| Removed|
|Menu.Menu Example Toolbar Shorthand.chromium.png|0| Removed|
|Menu.Menu Example Underlined Teams Dark.Clicks on 2nd item (active state styles).chromium.png|0| Removed|
|Menu.Menu Example Underlined Teams Dark.Hovers 2nd item (hover state styles).chromium.png|0| Removed|
|Menu.Menu Example Underlined Teams Dark.Navigates to next item (focus state styles).chromium.png|0| Removed|
|Menu.Menu Example Underlined Teams Dark.Navigates to previous item (active and focus state styles).chromium.png|0| Removed|
|Menu.Menu Example Underlined Teams Dark.chromium.png|0| Removed|
|Menu.Menu Example Underlined Teams High Contrast.Clicks on 2nd item (active state styles).chromium.png|0| Removed|
|Menu.Menu Example Underlined Teams High Contrast.Hovers 2nd item (hover state styles).chromium.png|0| Removed|
|Menu.Menu Example Underlined Teams High Contrast.Navigates to next item (focus state styles).chromium.png|0| Removed|
|Menu.Menu Example Underlined Teams High Contrast.Navigates to previous item (active and focus state styles).chromium.png|0| Removed|
|Menu.Menu Example Underlined Teams High Contrast.chromium.png|0| Removed|
|Menu.Menu Example Underlined Teams.Clicks on 2nd item (active state styles).chromium.png|0| Removed|
|Menu.Menu Example Underlined Teams.Hovers 2nd item (hover state styles).chromium.png|0| Removed|
|Menu.Menu Example Underlined Teams.Navigates to next item (focus state styles).chromium.png|0| Removed|
|Menu.Menu Example Underlined Teams.Navigates to previous item (active and focus state styles).chromium.png|0| Removed|
|Menu.Menu Example Underlined Teams.chromium.png|0| Removed|
|Menu.Menu Example Underlined.Clicks on 2nd item (active state styles).chromium.png|0| Removed|
|Menu.Menu Example Underlined.Hovers 2nd item (hover state styles).chromium.png|0| Removed|
|Menu.Menu Example Underlined.Navigates to next item (focus state styles).chromium.png|0| Removed|
|Menu.Menu Example Underlined.Navigates to previous item (active and focus state styles).chromium.png|0| Removed|
|Menu.Menu Example Underlined.chromium.png|0| Removed|
|Menu.Menu Example Vertical Pointing.Clicks on 2nd item (active state styles).chromium.png|0| Removed|
|Menu.Menu Example Vertical Pointing.Hovers 2nd item (hover state styles).chromium.png|0| Removed|
|Menu.Menu Example Vertical Pointing.Navigates to next item (focus state styles).chromium.png|0| Removed|
|Menu.Menu Example Vertical Pointing.Navigates to previous item (active and focus state styles).chromium.png|0| Removed|
|Menu.Menu Example Vertical Pointing.chromium.png|0| Removed|
|Menu.Menu Example Vertical Teams Dark.Clicks on 4th item (opens submenu).chromium.png|0| Removed|
|Menu.Menu Example Vertical Teams Dark.Hovers 1st item (hover state styles).chromium.png|0| Removed|
|Menu.Menu Example Vertical Teams Dark.Hovers 4th item (hover state styles).chromium.png|0| Removed|
|Menu.Menu Example Vertical Teams Dark.chromium.png|0| Removed|
|Menu.Menu Example Vertical Teams High Contrast.Clicks on 4th item (opens submenu).chromium.png|0| Removed|
|Menu.Menu Example Vertical Teams High Contrast.Hovers 1st item (hover state styles).chromium.png|0| Removed|
|Menu.Menu Example Vertical Teams High Contrast.Hovers 4th item (hover state styles).chromium.png|0| Removed|
|Menu.Menu Example Vertical Teams High Contrast.chromium.png|0| Removed|
|Menu.Menu Example Vertical Teams.Clicks on 4th item (opens submenu).chromium.png|0| Removed|
|Menu.Menu Example Vertical Teams.Hovers 1st item (hover state styles).chromium.png|0| Removed|
|Menu.Menu Example Vertical Teams.Hovers 4th item (hover state styles).chromium.png|0| Removed|
|Menu.Menu Example Vertical Teams.chromium.png|0| Removed|
|Menu.Menu Example Vertical.Clicks on 4th item (opens submenu).chromium.png|0| Removed|
|Menu.Menu Example Vertical.Hovers 1st item (hover state styles).chromium.png|0| Removed|
|Menu.Menu Example Vertical.Hovers 4th item (hover state styles).chromium.png|0| Removed|
|Menu.Menu Example Vertical.chromium.png|0| Removed|
|Menu.Menu Example With Icons.Clicks on 2nd item (active state styles).chromium.png|0| Removed|
|Menu.Menu Example With Icons.Hovers 2nd item (hover state styles).chromium.png|0| Removed|
|Menu.Menu Example With Icons.Navigates to next item (focus state styles).chromium.png|0| Removed|
|Menu.Menu Example With Icons.Navigates to previous item (active and focus state styles).chromium.png|0| Removed|
|Menu.Menu Example With Icons.chromium.png|0| Removed|
|Menu.Menu Example With Sub Menu Hover Teams Dark.Click 1st item, open menu.chromium.png|0| Removed|
|Menu.Menu Example With Sub Menu Hover Teams Dark.Hovers 2nd item, open submenu.chromium.png|0| Removed|
|Menu.Menu Example With Sub Menu Hover Teams Dark.chromium.png|0| Removed|
|Menu.Menu Example With Sub Menu Hover Teams High Contrast.Click 1st item, open menu.chromium.png|0| Removed|
|Menu.Menu Example With Sub Menu Hover Teams High Contrast.Hovers 2nd item, open submenu.chromium.png|0| Removed|
|Menu.Menu Example With Sub Menu Hover Teams High Contrast.chromium.png|0| Removed|
|Menu.Menu Example With Sub Menu Hover Teams.Click 1st item, open menu.chromium.png|0| Removed|
|Menu.Menu Example With Sub Menu Hover Teams.Hovers 2nd item, open submenu.chromium.png|0| Removed|
|Menu.Menu Example With Sub Menu Hover Teams.chromium.png|0| Removed|
|Menu.Menu Example With Sub Menu Hover.Click 1st item, open menu.chromium.png|0| Removed|
|Menu.Menu Example With Sub Menu Hover.Hovers 2nd item, open submenu.chromium.png|0| Removed|
|Menu.Menu Example With Sub Menu Hover.chromium.png|0| Removed|
|Menu.Menu Example With Sub Menu.Clicks on 2nd item (active state styles).chromium.png|0| Removed|
|Menu.Menu Example With Sub Menu.Hovers 2nd item (hover state styles).chromium.png|0| Removed|
|Menu.Menu Example With Sub Menu.Navigates to next item (focus state styles).chromium.png|0| Removed|
|Menu.Menu Example With Sub Menu.Navigates to previous item (active and focus state styles).chromium.png|0| Removed|
|Menu.Menu Example With Sub Menu.chromium.png|0| Removed|
|Menu.Menu Example With Tooltip Teams Dark.Hovers 1st item (show tooltip).chromium.png|0| Removed|
|Menu.Menu Example With Tooltip Teams Dark.Navigates to next item (shows tooltip).chromium.png|0| Removed|
|Menu.Menu Example With Tooltip Teams Dark.chromium.png|0| Removed|
|Menu.Menu Example With Tooltip Teams High Contrast.Hovers 1st item (show tooltip).chromium.png|0| Removed|
|Menu.Menu Example With Tooltip Teams High Contrast.Navigates to next item (shows tooltip).chromium.png|0| Removed|
|Menu.Menu Example With Tooltip Teams High Contrast.chromium.png|0| Removed|
|Menu.Menu Example With Tooltip Teams.Hovers 1st item (show tooltip).chromium.png|0| Removed|
|Menu.Menu Example With Tooltip Teams.Navigates to next item (shows tooltip).chromium.png|0| Removed|
|Menu.Menu Example With Tooltip Teams.chromium.png|0| Removed|
|Menu.Menu Example With Tooltip.Hovers 1st item (show tooltip).chromium.png|0| Removed|
|Menu.Menu Example With Tooltip.Navigates to next item (shows tooltip).chromium.png|0| Removed|
|Menu.Menu Example With Tooltip.chromium.png|0| Removed|
|Menu.Menu Example.Clicks on 2nd item (active state styles).chromium.png|0| Removed|
|Menu.Menu Example.Hovers 2nd item (hover state styles).chromium.png|0| Removed|
|Menu.Menu Example.Navigates to next item (focus state styles).chromium.png|0| Removed|
|Menu.Menu Example.Navigates to previous item (active and focus state styles).chromium.png|0| Removed|
|Menu.Menu Example.chromium.png|0| Removed|
|Menu.Menu Item Example Disabled.chromium.png|0| Removed|
|Menu.Menu Playground.chromium.png|0| Removed|
|Menu.Menu With Submenu Controlled Example.chromium.png|0| Removed|

</details>


  <details><summary><b>MenuButton</b> 5 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|MenuButton.MenuButtonExampleRtl.chromium.png|0| Added|
|MenuButton.MenuButtonOpenExample.chromium.png|0| Added|
|MenuButton.Menu Button Example Rtl.RTL- Shows menuButton.chromium.png|0| Removed|
|MenuButton.Menu Button Example Rtl.chromium.png|0| Removed|
|MenuButton.Menu Button Open Example.chromium.png|0| Removed|

</details>


  <details><summary><b>Pill</b> 20 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pill.PillActionableExample.chromium.png|0| Added|
|Pill.PillAppearanceExample.chromium.png|0| Added|
|Pill.PillExample.chromium.png|0| Added|
|Pill.PillExampleDisabled.chromium.png|0| Added|
|Pill.PillGroupExample.chromium.png|0| Added|
|Pill.PillIconExample.chromium.png|0| Added|
|Pill.PillImageExample.chromium.png|0| Added|
|Pill.PillRectangularExample.chromium.png|0| Added|
|Pill.PillSelectableExample.chromium.png|0| Added|
|Pill.PillSizesExample.chromium.png|0| Added|
|Pill.Pill Actionable Example.chromium.png|0| Removed|
|Pill.Pill Appearance Example.chromium.png|0| Removed|
|Pill.Pill Example Disabled.chromium.png|0| Removed|
|Pill.Pill Example.chromium.png|0| Removed|
|Pill.Pill Group Example.chromium.png|0| Removed|
|Pill.Pill Icon Example.chromium.png|0| Removed|
|Pill.Pill Image Example.chromium.png|0| Removed|
|Pill.Pill Rectangular Example.chromium.png|0| Removed|
|Pill.Pill Selectable Example.chromium.png|0| Removed|
|Pill.Pill Sizes Example.chromium.png|0| Removed|

</details>


  <details><summary><b>Popup</b> 66 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Popup.PopperExamplePositioning.chromium.png|0| Added|
|Popup.PopperExampleVisibilityModifiers.chromium.png|0| Added|
|Popup.PopupCloseButtonExample.chromium.png|0| Added|
|Popup.PopupControlledExample.chromium.png|0| Added|
|Popup.PopupControlledExampleTeams.chromium.png|0| Added|
|Popup.PopupControlledExampleTeamsDark.chromium.png|0| Added|
|Popup.PopupControlledExampleTeamsHighContrast.chromium.png|0| Added|
|Popup.PopupCustomTargetExample.chromium.png|0| Added|
|Popup.PopupExample.chromium.png|0| Added|
|Popup.PopupExampleAsync.chromium.png|0| Added|
|Popup.PopupExampleAutoSize.chromium.png|0| Added|
|Popup.PopupExampleContainerTransformed.chromium.png|0| Added|
|Popup.PopupExampleInline.chromium.png|0| Added|
|Popup.PopupExampleNested.chromium.png|0| Added|
|Popup.PopupExampleOffset.chromium.png|0| Added|
|Popup.PopupExamplePointerMargin.chromium.png|0| Added|
|Popup.PopupExamplePointerOffset.chromium.png|0| Added|
|Popup.PopupExamplePointing.chromium.png|0| Added|
|Popup.PopupExamplePosition.chromium.png|0| Added|
|Popup.PopupExampleRtl.chromium.png|0| Added|
|Popup.PopupExampleTeams.chromium.png|0| Added|
|Popup.PopupExampleTeamsDark.chromium.png|0| Added|
|Popup.PopupExampleTeamsHighContrast.chromium.png|0| Added|
|Popup.PopupFocusTrapExample.chromium.png|0| Added|
|Popup.PopupScrollExample.chromium.png|0| Added|
|Popup.Popper Example Positioning.chromium.png|0| Removed|
|Popup.Popper Example Visibility Modifiers.Opened a popup on second message.chromium.png|0| Removed|
|Popup.Popper Example Visibility Modifiers.chromium.png|0| Removed|
|Popup.Popper Example Visibility Modifiers.has  [data-popper-escaped]  when the popper escapes the reference element's boundary.chromium.png|0| Removed|
|Popup.Popper Example Visibility Modifiers.has  [data-popper-is-intersecting]  when the popover intersects boundaries.chromium.png|0| Removed|
|Popup.Popper Example Visibility Modifiers.has  [data-popper-reference-hidden]  when the reference is hidden.chromium.png|0| Removed|
|Popup.Popup Close Button Example.chromium.png|0| Removed|
|Popup.Popup Controlled Example Teams Dark.chromium.png|0| Removed|
|Popup.Popup Controlled Example Teams High Contrast.chromium.png|0| Removed|
|Popup.Popup Controlled Example Teams.chromium.png|0| Removed|
|Popup.Popup Controlled Example.chromium.png|0| Removed|
|Popup.Popup Custom Target Example.chromium.png|0| Removed|
|Popup.Popup Example Async.chromium.png|0| Removed|
|Popup.Popup Example Auto Size.chromium.png|0| Removed|
|Popup.Popup Example Container Transformed.chromium.png|0| Removed|
|Popup.Popup Example Inline.chromium.png|0| Removed|
|Popup.Popup Example Nested.chromium.png|0| Removed|
|Popup.Popup Example Offset.Opens a popup.chromium.png|0| Removed|
|Popup.Popup Example Offset.chromium.png|0| Removed|
|Popup.Popup Example Pointer Margin.chromium.png|0| Removed|
|Popup.Popup Example Pointer Offset.chromium.png|0| Removed|
|Popup.Popup Example Pointing.chromium.png|0| Removed|
|Popup.Popup Example Position.chromium.png|0| Removed|
|Popup.Popup Example Rtl.RTL- Shows popup.chromium.png|0| Removed|
|Popup.Popup Example Rtl.chromium.png|0| Removed|
|Popup.Popup Example Teams Dark.Shows popup.chromium.png|0| Removed|
|Popup.Popup Example Teams Dark.chromium.png|0| Removed|
|Popup.Popup Example Teams High Contrast.Shows popup.chromium.png|0| Removed|
|Popup.Popup Example Teams High Contrast.chromium.png|0| Removed|
|Popup.Popup Example Teams.Shows popup.chromium.png|0| Removed|
|Popup.Popup Example Teams.chromium.png|0| Removed|
|Popup.Popup Example.Shows popup.chromium.png|0| Removed|
|Popup.Popup Example.chromium.png|0| Removed|
|Popup.Popup Focus Trap Example.chromium.png|0| Removed|
|Popup.Popup Scroll Example.Click on trigger button for controlled popup with manual focus.chromium.png|0| Removed|
|Popup.Popup Scroll Example.Click on trigger button for inline controlled popup with manual focus.chromium.png|0| Removed|
|Popup.Popup Scroll Example.Click on trigger button for popup with trap focus.chromium.png|0| Removed|
|Popup.Popup Scroll Example.Scroll to trigger button for controlled popup with manual focus.chromium.png|0| Removed|
|Popup.Popup Scroll Example.Scroll to trigger button for inline controlled popup with manual focus.chromium.png|0| Removed|
|Popup.Popup Scroll Example.Scroll to trigger button for popup with trap focus.chromium.png|0| Removed|
|Popup.Popup Scroll Example.chromium.png|0| Removed|

</details>


  <details><summary><b>Portal</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Portal.PortalExampleOpen.chromium.png|0| Added|
|Portal.Portal Example Open.chromium.png|0| Removed|

</details>


  <details><summary><b>Provider</b> 15 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Provider.ProviderExampleAnimation.chromium.png|0| Added|
|Provider.ProviderExampleFocusBorderShorthand.chromium.png|0| Added|
|Provider.ProviderExampleScrollbar.chromium.png|0| Added|
|Provider.ProviderExampleShorthand.chromium.png|0| Added|
|Provider.ProviderExampleTarget.chromium.png|0| Added|
|Provider.ProviderExampleTargetFrame.chromium.png|0| Added|
|Provider.ProviderRtlExample.chromium.png|0| Added|
|Provider.Provider Example Animation.chromium.png|0| Removed|
|Provider.Provider Example Focus Border Shorthand.Show overridden border focused styles.chromium.png|0| Removed|
|Provider.Provider Example Focus Border Shorthand.chromium.png|0| Removed|
|Provider.Provider Example Scrollbar.chromium.png|0| Removed|
|Provider.Provider Example Shorthand.chromium.png|0| Removed|
|Provider.Provider Example Target Frame.chromium.png|0| Removed|
|Provider.Provider Example Target.chromium.png|0| Removed|
|Provider.Provider Rtl Example.chromium.png|0| Removed|

</details>


  <details><summary><b>RadioGroup</b> 20 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|RadioGroup.RadioGroupColorPickerExample.chromium.png|0| Added|
|RadioGroup.RadioGroupCustomExample.chromium.png|0| Added|
|RadioGroup.RadioGroupExampleRtl.chromium.png|0| Added|
|RadioGroup.RadioGroupItemExample.chromium.png|0| Added|
|RadioGroup.RadioGroupItemExampleCheckedShorthand.chromium.png|0| Added|
|RadioGroup.RadioGroupItemExampleDisabledShorthand.chromium.png|0| Added|
|RadioGroup.RadioGroupVerticalExample.chromium.png|0| Added|
|RadioGroup.RadioGroupVerticalExampleTeams.chromium.png|0| Added|
|RadioGroup.RadioGroupVerticalExampleTeamsDark.chromium.png|0| Added|
|RadioGroup.RadioGroupVerticalExampleTeamsHighContrast.chromium.png|0| Added|
|RadioGroup.Radio Group Color Picker Example.chromium.png|0| Removed|
|RadioGroup.Radio Group Custom Example.chromium.png|0| Removed|
|RadioGroup.Radio Group Example Rtl.chromium.png|0| Removed|
|RadioGroup.Radio Group Item Example Checked Shorthand.chromium.png|0| Removed|
|RadioGroup.Radio Group Item Example Disabled Shorthand.chromium.png|0| Removed|
|RadioGroup.Radio Group Item Example.chromium.png|0| Removed|
|RadioGroup.Radio Group Vertical Example Teams Dark.chromium.png|0| Removed|
|RadioGroup.Radio Group Vertical Example Teams High Contrast.chromium.png|0| Removed|
|RadioGroup.Radio Group Vertical Example Teams.chromium.png|0| Removed|
|RadioGroup.Radio Group Vertical Example.chromium.png|0| Removed|

</details>


  <details><summary><b>Reaction</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Reaction.ReactionExample.chromium.png|0| Added|
|Reaction.ReactionGroupExample.chromium.png|0| Added|
|Reaction.ReactionGroupExampleRtl.chromium.png|0| Added|
|Reaction.Reaction Example.chromium.png|0| Removed|
|Reaction.Reaction Group Example Rtl.chromium.png|0| Removed|
|Reaction.Reaction Group Example.chromium.png|0| Removed|

</details>


  <details><summary><b>Segment</b> 14 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Segment.SegmentExample.chromium.png|0| Added|
|Segment.SegmentExampleColor.chromium.png|0| Added|
|Segment.SegmentExampleDisabled.chromium.png|0| Added|
|Segment.SegmentExampleDisabledShorthand.chromium.png|0| Added|
|Segment.SegmentExampleInvertedShorthand.chromium.png|0| Added|
|Segment.SegmentExampleRtl.chromium.png|0| Added|
|Segment.SegmentExampleShorthand.chromium.png|0| Added|
|Segment.Segment Example Color.chromium.png|0| Removed|
|Segment.Segment Example Disabled Shorthand.chromium.png|0| Removed|
|Segment.Segment Example Disabled.chromium.png|0| Removed|
|Segment.Segment Example Inverted Shorthand.chromium.png|0| Removed|
|Segment.Segment Example Rtl.chromium.png|0| Removed|
|Segment.Segment Example Shorthand.chromium.png|0| Removed|
|Segment.Segment Example.chromium.png|0| Removed|

</details>


  <details><summary><b>Skeleton</b> 10 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Skeleton.SkeletonExampleAnimations.chromium.png|0| Added|
|Skeleton.SkeletonExampleCard.chromium.png|0| Added|
|Skeleton.SkeletonExampleComponents.chromium.png|0| Added|
|Skeleton.SkeletonExampleDefault.chromium.png|0| Added|
|Skeleton.SkeletonExampleList.chromium.png|0| Added|
|Skeleton.Skeleton Example Animations.chromium.png|0| Removed|
|Skeleton.Skeleton Example Card.chromium.png|0| Removed|
|Skeleton.Skeleton Example Components.chromium.png|0| Removed|
|Skeleton.Skeleton Example Default.chromium.png|0| Removed|
|Skeleton.Skeleton Example List.chromium.png|0| Removed|

</details>


  <details><summary><b>Slider</b> 34 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Slider.SliderExampleActionShorthand.chromium.png|0| Added|
|Slider.SliderExampleDisabledShorthand.chromium.png|0| Added|
|Slider.SliderExampleFluidShorthand.chromium.png|0| Added|
|Slider.SliderExampleFormShorthand.chromium.png|0| Added|
|Slider.SliderExampleRtl.chromium.png|0| Added|
|Slider.SliderExampleRtlTeams.chromium.png|0| Added|
|Slider.SliderExampleRtlTeamsDark.chromium.png|0| Added|
|Slider.SliderExampleRtlTeamsHighContrast.chromium.png|0| Added|
|Slider.SliderExampleShorthand.chromium.png|0| Added|
|Slider.SliderExampleShorthandTeams.chromium.png|0| Added|
|Slider.SliderExampleShorthandTeamsDark.chromium.png|0| Added|
|Slider.SliderExampleShorthandTeamsHighContrast.chromium.png|0| Added|
|Slider.SliderExampleVerticalShorthand.chromium.png|0| Added|
|Slider.Slider Example Action Shorthand.chromium.png|0| Removed|
|Slider.Slider Example Disabled Shorthand.chromium.png|0| Removed|
|Slider.Slider Example Fluid Shorthand.chromium.png|0| Removed|
|Slider.Slider Example Form Shorthand.chromium.png|0| Removed|
|Slider.Slider Example Rtl Teams Dark.Navigates to the right with the up arrow key.chromium.png|0| Removed|
|Slider.Slider Example Rtl Teams Dark.chromium.png|0| Removed|
|Slider.Slider Example Rtl Teams High Contrast.Navigates to the right with the up arrow key.chromium.png|0| Removed|
|Slider.Slider Example Rtl Teams High Contrast.chromium.png|0| Removed|
|Slider.Slider Example Rtl Teams.Navigates to the right with the up arrow key.chromium.png|0| Removed|
|Slider.Slider Example Rtl Teams.chromium.png|0| Removed|
|Slider.Slider Example Rtl.Navigates to the right with the up arrow key.chromium.png|0| Removed|
|Slider.Slider Example Rtl.chromium.png|0| Removed|
|Slider.Slider Example Shorthand Teams Dark.Navigates to the right with the up arrow key.chromium.png|0| Removed|
|Slider.Slider Example Shorthand Teams Dark.chromium.png|0| Removed|
|Slider.Slider Example Shorthand Teams High Contrast.Navigates to the right with the up arrow key.chromium.png|0| Removed|
|Slider.Slider Example Shorthand Teams High Contrast.chromium.png|0| Removed|
|Slider.Slider Example Shorthand Teams.Navigates to the right with the up arrow key.chromium.png|0| Removed|
|Slider.Slider Example Shorthand Teams.chromium.png|0| Removed|
|Slider.Slider Example Shorthand.Navigates to the right with the up arrow key.chromium.png|0| Removed|
|Slider.Slider Example Shorthand.chromium.png|0| Removed|
|Slider.Slider Example Vertical Shorthand.chromium.png|0| Removed|

</details>


  <details><summary><b>SplitButton</b> 26 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SplitButton.SplitButtonExampleDisabledShorthand.chromium.png|0| Added|
|SplitButton.SplitButtonExampleFlat.chromium.png|0| Added|
|SplitButton.SplitButtonExamplePrimaryShorthand.chromium.png|0| Added|
|SplitButton.SplitButtonExampleShorthand.chromium.png|0| Added|
|SplitButton.SplitButtonExampleSmallContainer.chromium.png|0| Added|
|SplitButton.SplitButtonExampleSmallContainerTeams.chromium.png|0| Added|
|SplitButton.SplitButtonExampleSmallContainerTeamsDark.chromium.png|0| Added|
|SplitButton.SplitButtonExampleSmallContainerTeamsHighContrast.chromium.png|0| Added|
|SplitButton.SplitButtonExampleSmallShorthand.chromium.png|0| Added|
|SplitButton.SplitButtonExampleToggleButtonShorthand.chromium.png|0| Added|
|SplitButton.SplitButtonIconAndContentExampleShorthand.chromium.png|0| Added|
|SplitButton.SplitButtonMainOptionChangeExample.chromium.png|0| Added|
|SplitButton.SplitButtonPositioningExampleShorthand.chromium.png|0| Added|
|SplitButton.Split Button Example Disabled Shorthand.chromium.png|0| Removed|
|SplitButton.Split Button Example Flat.chromium.png|0| Removed|
|SplitButton.Split Button Example Shorthand.chromium.png|0| Removed|
|SplitButton.Split Button Example Small Container Teams Dark.chromium.png|0| Removed|
|SplitButton.Split Button Example Small Container Teams High Contrast.chromium.png|0| Removed|
|SplitButton.Split Button Example Small Container Teams.chromium.png|0| Removed|
|SplitButton.Split Button Example Small Container.chromium.png|0| Removed|
|SplitButton.Split Button Example Small Shorthand.chromium.png|0| Removed|
|SplitButton.Split Button Example Toggle Button Shorthand.chromium.png|0| Removed|
|SplitButton.Split Button Icon And Content Example Shorthand.chromium.png|0| Removed|
|SplitButton.Split Button Main Option Change Example.chromium.png|0| Removed|
|SplitButton.Split Button Positioning Example Shorthand.Open menu.chromium.png|0| Removed|
|SplitButton.Split Button Positioning Example Shorthand.chromium.png|0| Removed|

</details>


  <details><summary><b>Status</b> 18 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Status.StatusColorExampleShorthand.chromium.png|0| Added|
|Status.StatusCustomExampleShorthand.chromium.png|0| Added|
|Status.StatusExampleShorthand.chromium.png|0| Added|
|Status.StatusIconExampleShorthand.chromium.png|0| Added|
|Status.StatusSizeExampleShorthand.chromium.png|0| Added|
|Status.StatusTypeExampleShorthand.chromium.png|0| Added|
|Status.StatusTypeExampleShorthandTeams.chromium.png|0| Added|
|Status.StatusTypeExampleShorthandTeamsDark.chromium.png|0| Added|
|Status.StatusTypeExampleShorthandTeamsHighContrast.chromium.png|0| Added|
|Status.Status Color Example Shorthand.chromium.png|0| Removed|
|Status.Status Custom Example Shorthand.chromium.png|0| Removed|
|Status.Status Example Shorthand.chromium.png|0| Removed|
|Status.Status Icon Example Shorthand.chromium.png|0| Removed|
|Status.Status Size Example Shorthand.chromium.png|0| Removed|
|Status.Status Type Example Shorthand Teams Dark.chromium.png|0| Removed|
|Status.Status Type Example Shorthand Teams High Contrast.chromium.png|0| Removed|
|Status.Status Type Example Shorthand Teams.chromium.png|0| Removed|
|Status.Status Type Example Shorthand.chromium.png|0| Removed|

</details>


  <details><summary><b>SvgIcon</b> 72 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SvgIcon.SvgIconExample.chromium.png|0| Added|
|SvgIcon.SvgIconExampleCircular.chromium.png|0| Added|
|SvgIcon.SvgIconExampleCircularTeams.chromium.png|0| Added|
|SvgIcon.SvgIconExampleCircularTeamsDark.chromium.png|0| Added|
|SvgIcon.SvgIconExampleCircularTeamsHighContrast.chromium.png|0| Added|
|SvgIcon.SvgIconExampleColor.chromium.png|0| Added|
|SvgIcon.SvgIconExampleColorTeams.chromium.png|0| Added|
|SvgIcon.SvgIconExampleColorTeamsDark.chromium.png|0| Added|
|SvgIcon.SvgIconExampleColorTeamsHighContrast.chromium.png|0| Added|
|SvgIcon.SvgIconExampleDisabled.chromium.png|0| Added|
|SvgIcon.SvgIconExampleDisabledTeams.chromium.png|0| Added|
|SvgIcon.SvgIconExampleDisabledTeamsDark.chromium.png|0| Added|
|SvgIcon.SvgIconExampleDisabledTeamsHighContrast.chromium.png|0| Added|
|SvgIcon.SvgIconExampleRotate.chromium.png|0| Added|
|SvgIcon.SvgIconExampleRotateRtl.chromium.png|0| Added|
|SvgIcon.SvgIconExampleRtl.chromium.png|0| Added|
|SvgIcon.SvgIconExampleSize.chromium.png|0| Added|
|SvgIcon.SvgIconExampleSizeTeams.chromium.png|0| Added|
|SvgIcon.SvgIconExampleSizeTeamsDark.chromium.png|0| Added|
|SvgIcon.SvgIconExampleSizeTeamsHighContrast.chromium.png|0| Added|
|SvgIcon.SvgIconExampleSpace.chromium.png|0| Added|
|SvgIcon.SvgIconExampleSpaceTeams.chromium.png|0| Added|
|SvgIcon.SvgIconExampleSpaceTeamsDark.chromium.png|0| Added|
|SvgIcon.SvgIconExampleSpaceTeamsHighContrast.chromium.png|0| Added|
|SvgIcon.SvgIconExampleTeams.chromium.png|0| Added|
|SvgIcon.SvgIconExampleTeamsDark.chromium.png|0| Added|
|SvgIcon.SvgIconExampleTeamsHighContrast.chromium.png|0| Added|
|SvgIcon.SvgIconSetExampleShorthand.chromium.png|0| Added|
|SvgIcon.SvgIconSetExampleShorthandTeams.chromium.png|0| Added|
|SvgIcon.SvgIconSetExampleShorthandTeamsDark.chromium.png|0| Added|
|SvgIcon.SvgIconSetExampleShorthandTeamsHighContrast.chromium.png|0| Added|
|SvgIcon.SvgPlayground.chromium.png|0| Added|
|SvgIcon.SvgconExampleBordered.chromium.png|0| Added|
|SvgIcon.SvgconExampleBorderedTeams.chromium.png|0| Added|
|SvgIcon.SvgconExampleBorderedTeamsDark.chromium.png|0| Added|
|SvgIcon.SvgconExampleBorderedTeamsHighContrast.chromium.png|0| Added|
|SvgIcon.Svg Icon Example Circular Teams Dark.chromium.png|0| Removed|
|SvgIcon.Svg Icon Example Circular Teams High Contrast.chromium.png|0| Removed|
|SvgIcon.Svg Icon Example Circular Teams.chromium.png|0| Removed|
|SvgIcon.Svg Icon Example Circular.chromium.png|0| Removed|
|SvgIcon.Svg Icon Example Color Teams Dark.chromium.png|0| Removed|
|SvgIcon.Svg Icon Example Color Teams High Contrast.chromium.png|0| Removed|
|SvgIcon.Svg Icon Example Color Teams.chromium.png|0| Removed|
|SvgIcon.Svg Icon Example Color.chromium.png|0| Removed|
|SvgIcon.Svg Icon Example Disabled Teams Dark.chromium.png|0| Removed|
|SvgIcon.Svg Icon Example Disabled Teams High Contrast.chromium.png|0| Removed|
|SvgIcon.Svg Icon Example Disabled Teams.chromium.png|0| Removed|
|SvgIcon.Svg Icon Example Disabled.chromium.png|0| Removed|
|SvgIcon.Svg Icon Example Rotate Rtl.chromium.png|0| Removed|
|SvgIcon.Svg Icon Example Rotate.chromium.png|0| Removed|
|SvgIcon.Svg Icon Example Rtl.chromium.png|0| Removed|
|SvgIcon.Svg Icon Example Size Teams Dark.chromium.png|0| Removed|
|SvgIcon.Svg Icon Example Size Teams High Contrast.chromium.png|0| Removed|
|SvgIcon.Svg Icon Example Size Teams.chromium.png|0| Removed|
|SvgIcon.Svg Icon Example Size.chromium.png|0| Removed|
|SvgIcon.Svg Icon Example Space Teams Dark.chromium.png|0| Removed|
|SvgIcon.Svg Icon Example Space Teams High Contrast.chromium.png|0| Removed|
|SvgIcon.Svg Icon Example Space Teams.chromium.png|0| Removed|
|SvgIcon.Svg Icon Example Space.chromium.png|0| Removed|
|SvgIcon.Svg Icon Example Teams Dark.chromium.png|0| Removed|
|SvgIcon.Svg Icon Example Teams High Contrast.chromium.png|0| Removed|
|SvgIcon.Svg Icon Example Teams.chromium.png|0| Removed|
|SvgIcon.Svg Icon Example.chromium.png|0| Removed|
|SvgIcon.Svg Icon Set Example Shorthand Teams Dark.chromium.png|0| Removed|
|SvgIcon.Svg Icon Set Example Shorthand Teams High Contrast.chromium.png|0| Removed|
|SvgIcon.Svg Icon Set Example Shorthand Teams.chromium.png|0| Removed|
|SvgIcon.Svg Icon Set Example Shorthand.chromium.png|0| Removed|
|SvgIcon.Svg Playground.chromium.png|0| Removed|
|SvgIcon.Svgcon Example Bordered Teams Dark.chromium.png|0| Removed|
|SvgIcon.Svgcon Example Bordered Teams High Contrast.chromium.png|0| Removed|
|SvgIcon.Svgcon Example Bordered Teams.chromium.png|0| Removed|
|SvgIcon.Svgcon Example Bordered.chromium.png|0| Removed|

</details>


  <details><summary><b>Table</b> 14 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table.StaticTable.chromium.png|0| Added|
|Table.StaticTableCompact.chromium.png|0| Added|
|Table.StaticTableTeams.chromium.png|0| Added|
|Table.StaticTableTeamsDark.chromium.png|0| Added|
|Table.StaticTableTeamsHighContrast.chromium.png|0| Added|
|Table.TableExampleStatic.chromium.png|0| Added|
|Table.TableExampleStaticShorthand.chromium.png|0| Added|
|Table.Static Table Compact.chromium.png|0| Removed|
|Table.Static Table Teams Dark.chromium.png|0| Removed|
|Table.Static Table Teams High Contrast.chromium.png|0| Removed|
|Table.Static Table Teams.chromium.png|0| Removed|
|Table.Static Table.chromium.png|0| Removed|
|Table.Table Example Static Shorthand.chromium.png|0| Removed|
|Table.Table Example Static.chromium.png|0| Removed|

</details>


  <details><summary><b>Text</b> 46 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Text.HeaderExampleAlign.chromium.png|0| Added|
|Text.TextExampleAtMention.chromium.png|0| Added|
|Text.TextExampleAtMentionShorthand.chromium.png|0| Added|
|Text.TextExampleColor.chromium.png|0| Added|
|Text.TextExampleDisabled.chromium.png|0| Added|
|Text.TextExampleDisabledShorthand.chromium.png|0| Added|
|Text.TextExampleError.chromium.png|0| Added|
|Text.TextExampleErrorShorthand.chromium.png|0| Added|
|Text.TextExampleImportant.chromium.png|0| Added|
|Text.TextExampleImportantShorthand.chromium.png|0| Added|
|Text.TextExampleRtl.chromium.png|0| Added|
|Text.TextExampleSuccess.chromium.png|0| Added|
|Text.TextExampleSuccessShorthand.chromium.png|0| Added|
|Text.TextExampleTemporary.chromium.png|0| Added|
|Text.TextExampleTemporaryShorthand.chromium.png|0| Added|
|Text.TextExampleTimestamp.chromium.png|0| Added|
|Text.TextExampleTimestampShorthand.chromium.png|0| Added|
|Text.TextExampleTruncated.chromium.png|0| Added|
|Text.TextExampleTruncatedShorthand.chromium.png|0| Added|
|Text.TextSizesExample.chromium.png|0| Added|
|Text.TextSizesExampleShorthand.chromium.png|0| Added|
|Text.TextWeightsExample.chromium.png|0| Added|
|Text.TextWeightsExampleShorthand.chromium.png|0| Added|
|Text.Header Example Align.chromium.png|0| Removed|
|Text.Text Example At Mention Shorthand.chromium.png|0| Removed|
|Text.Text Example At Mention.chromium.png|0| Removed|
|Text.Text Example Color.chromium.png|0| Removed|
|Text.Text Example Disabled Shorthand.chromium.png|0| Removed|
|Text.Text Example Disabled.chromium.png|0| Removed|
|Text.Text Example Error Shorthand.chromium.png|0| Removed|
|Text.Text Example Error.chromium.png|0| Removed|
|Text.Text Example Important Shorthand.chromium.png|0| Removed|
|Text.Text Example Important.chromium.png|0| Removed|
|Text.Text Example Rtl.chromium.png|0| Removed|
|Text.Text Example Success Shorthand.chromium.png|0| Removed|
|Text.Text Example Success.chromium.png|0| Removed|
|Text.Text Example Temporary Shorthand.chromium.png|0| Removed|
|Text.Text Example Temporary.chromium.png|0| Removed|
|Text.Text Example Timestamp Shorthand.chromium.png|0| Removed|
|Text.Text Example Timestamp.chromium.png|0| Removed|
|Text.Text Example Truncated Shorthand.chromium.png|0| Removed|
|Text.Text Example Truncated.chromium.png|0| Removed|
|Text.Text Sizes Example Shorthand.chromium.png|0| Removed|
|Text.Text Sizes Example.chromium.png|0| Removed|
|Text.Text Weights Example Shorthand.chromium.png|0| Removed|
|Text.Text Weights Example.chromium.png|0| Removed|

</details>


  <details><summary><b>TextArea</b> 32 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|TextArea.TextAreaDisabledExample.chromium.png|0| Added|
|TextArea.TextAreaExample.chromium.png|0| Added|
|TextArea.TextAreaExampleFluid.chromium.png|0| Added|
|TextArea.TextAreaExampleHeight.chromium.png|0| Added|
|TextArea.TextAreaExampleInverted.chromium.png|0| Added|
|TextArea.TextAreaExampleInvertedTeams.chromium.png|0| Added|
|TextArea.TextAreaExampleInvertedTeamsDark.chromium.png|0| Added|
|TextArea.TextAreaExampleInvertedTeamsHighContrast.chromium.png|0| Added|
|TextArea.TextAreaExampleMaxLength.chromium.png|0| Added|
|TextArea.TextAreaExampleResize.chromium.png|0| Added|
|TextArea.TextAreaExampleTeams.chromium.png|0| Added|
|TextArea.TextAreaExampleTeamsDark.chromium.png|0| Added|
|TextArea.TextAreaExampleTeamsHighContrast.chromium.png|0| Added|
|TextArea.TextAreaValueExample.chromium.png|0| Added|
|TextArea.Text Area Disabled Example.chromium.png|0| Removed|
|TextArea.Text Area Example Fluid.chromium.png|0| Removed|
|TextArea.Text Area Example Height.chromium.png|0| Removed|
|TextArea.Text Area Example Inverted Teams Dark.chromium.png|0| Removed|
|TextArea.Text Area Example Inverted Teams High Contrast.chromium.png|0| Removed|
|TextArea.Text Area Example Inverted Teams.chromium.png|0| Removed|
|TextArea.Text Area Example Inverted.chromium.png|0| Removed|
|TextArea.Text Area Example Max Length.chromium.png|0| Removed|
|TextArea.Text Area Example Resize.chromium.png|0| Removed|
|TextArea.Text Area Example Teams Dark.Can be focused.chromium.png|0| Removed|
|TextArea.Text Area Example Teams Dark.chromium.png|0| Removed|
|TextArea.Text Area Example Teams High Contrast.Can be focused.chromium.png|0| Removed|
|TextArea.Text Area Example Teams High Contrast.chromium.png|0| Removed|
|TextArea.Text Area Example Teams.Can be focused.chromium.png|0| Removed|
|TextArea.Text Area Example Teams.chromium.png|0| Removed|
|TextArea.Text Area Example.Can be focused.chromium.png|0| Removed|
|TextArea.Text Area Example.chromium.png|0| Removed|
|TextArea.Text Area Value Example.chromium.png|0| Removed|

</details>


  <details><summary><b>Toolbar</b> 76 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Toolbar.ToolbarExample.chromium.png|0| Added|
|Toolbar.ToolbarExampleActionPopupInMenu.chromium.png|0| Added|
|Toolbar.ToolbarExampleActionPopupInMenuTeams.chromium.png|0| Added|
|Toolbar.ToolbarExampleActionPopupInMenuTeamsDark.chromium.png|0| Added|
|Toolbar.ToolbarExampleActionPopupInMenuTeamsHighContrast.chromium.png|0| Added|
|Toolbar.ToolbarExampleCustomContentShorthand.chromium.png|0| Added|
|Toolbar.ToolbarExampleEditorShorthand.chromium.png|0| Added|
|Toolbar.ToolbarExampleEditorShorthandTeams.chromium.png|0| Added|
|Toolbar.ToolbarExampleEditorShorthandTeamsDark.chromium.png|0| Added|
|Toolbar.ToolbarExampleEditorShorthandTeamsHighContrast.chromium.png|0| Added|
|Toolbar.ToolbarExampleMenuItemToggle.chromium.png|0| Added|
|Toolbar.ToolbarExampleMenuRadioGroup.chromium.png|0| Added|
|Toolbar.ToolbarExampleMenuShorthand.chromium.png|0| Added|
|Toolbar.ToolbarExampleMenuWithSubmenuShorthand.chromium.png|0| Added|
|Toolbar.ToolbarExampleMenuWithSubmenuShorthandTeams.chromium.png|0| Added|
|Toolbar.ToolbarExampleMenuWithSubmenuShorthandTeamsDark.chromium.png|0| Added|
|Toolbar.ToolbarExampleMenuWithSubmenuShorthandTeamsHighContrast.chromium.png|0| Added|
|Toolbar.ToolbarExampleOverflow.chromium.png|0| Added|
|Toolbar.ToolbarExampleOverflowPositioningShorthand.chromium.png|0| Added|
|Toolbar.ToolbarExampleOverflowPositioningShorthandRtl.chromium.png|0| Added|
|Toolbar.ToolbarExamplePopupInMenu.chromium.png|0| Added|
|Toolbar.ToolbarExamplePopupInMenuTeams.chromium.png|0| Added|
|Toolbar.ToolbarExamplePopupInMenuTeamsDark.chromium.png|0| Added|
|Toolbar.ToolbarExamplePopupInMenuTeamsHighContrast.chromium.png|0| Added|
|Toolbar.ToolbarExamplePopupShorthand.chromium.png|0| Added|
|Toolbar.ToolbarExamplePopupShorthandTeams.chromium.png|0| Added|
|Toolbar.ToolbarExamplePopupShorthandTeamsDark.chromium.png|0| Added|
|Toolbar.ToolbarExamplePopupShorthandTeamsHighContrast.chromium.png|0| Added|
|Toolbar.ToolbarExampleRadioGroupShorthand.chromium.png|0| Added|
|Toolbar.ToolbarExampleRadioGroupShorthandTeams.chromium.png|0| Added|
|Toolbar.ToolbarExampleRadioGroupShorthandTeamsDark.chromium.png|0| Added|
|Toolbar.ToolbarExampleRadioGroupShorthandTeamsHighContrast.chromium.png|0| Added|
|Toolbar.ToolbarExampleShorthand.chromium.png|0| Added|
|Toolbar.ToolbarExampleShorthandTeams.chromium.png|0| Added|
|Toolbar.ToolbarExampleShorthandTeamsDark.chromium.png|0| Added|
|Toolbar.ToolbarExampleShorthandTeamsHighContrast.chromium.png|0| Added|
|Toolbar.ToolbarExampleVariables.chromium.png|0| Added|
|Toolbar.Toolbar Example Action Popup In Menu Teams Dark.chromium.png|0| Removed|
|Toolbar.Toolbar Example Action Popup In Menu Teams High Contrast.chromium.png|0| Removed|
|Toolbar.Toolbar Example Action Popup In Menu Teams.chromium.png|0| Removed|
|Toolbar.Toolbar Example Action Popup In Menu.chromium.png|0| Removed|
|Toolbar.Toolbar Example Custom Content Shorthand.chromium.png|0| Removed|
|Toolbar.Toolbar Example Editor Shorthand Teams Dark.chromium.png|0| Removed|
|Toolbar.Toolbar Example Editor Shorthand Teams High Contrast.chromium.png|0| Removed|
|Toolbar.Toolbar Example Editor Shorthand Teams.chromium.png|0| Removed|
|Toolbar.Toolbar Example Editor Shorthand.chromium.png|0| Removed|
|Toolbar.Toolbar Example Menu Item Toggle.chromium.png|0| Removed|
|Toolbar.Toolbar Example Menu Radio Group.chromium.png|0| Removed|
|Toolbar.Toolbar Example Menu Shorthand.chromium.png|0| Removed|
|Toolbar.Toolbar Example Menu With Submenu Shorthand Teams Dark.chromium.png|0| Removed|
|Toolbar.Toolbar Example Menu With Submenu Shorthand Teams High Contrast.chromium.png|0| Removed|
|Toolbar.Toolbar Example Menu With Submenu Shorthand Teams.chromium.png|0| Removed|
|Toolbar.Toolbar Example Menu With Submenu Shorthand.chromium.png|0| Removed|
|Toolbar.Toolbar Example Overflow Positioning Shorthand Rtl.RTL- Overflow item is properly positioned.chromium.png|0| Removed|
|Toolbar.Toolbar Example Overflow Positioning Shorthand Rtl.chromium.png|0| Removed|
|Toolbar.Toolbar Example Overflow Positioning Shorthand.Overflow item is properly positioned.chromium.png|0| Removed|
|Toolbar.Toolbar Example Overflow Positioning Shorthand.chromium.png|0| Removed|
|Toolbar.Toolbar Example Overflow.chromium.png|0| Removed|
|Toolbar.Toolbar Example Popup In Menu Teams Dark.chromium.png|0| Removed|
|Toolbar.Toolbar Example Popup In Menu Teams High Contrast.chromium.png|0| Removed|
|Toolbar.Toolbar Example Popup In Menu Teams.chromium.png|0| Removed|
|Toolbar.Toolbar Example Popup In Menu.chromium.png|0| Removed|
|Toolbar.Toolbar Example Popup Shorthand Teams Dark.chromium.png|0| Removed|
|Toolbar.Toolbar Example Popup Shorthand Teams High Contrast.chromium.png|0| Removed|
|Toolbar.Toolbar Example Popup Shorthand Teams.chromium.png|0| Removed|
|Toolbar.Toolbar Example Popup Shorthand.chromium.png|0| Removed|
|Toolbar.Toolbar Example Radio Group Shorthand Teams Dark.chromium.png|0| Removed|
|Toolbar.Toolbar Example Radio Group Shorthand Teams High Contrast.chromium.png|0| Removed|
|Toolbar.Toolbar Example Radio Group Shorthand Teams.chromium.png|0| Removed|
|Toolbar.Toolbar Example Radio Group Shorthand.chromium.png|0| Removed|
|Toolbar.Toolbar Example Shorthand Teams Dark.chromium.png|0| Removed|
|Toolbar.Toolbar Example Shorthand Teams High Contrast.chromium.png|0| Removed|
|Toolbar.Toolbar Example Shorthand Teams.chromium.png|0| Removed|
|Toolbar.Toolbar Example Shorthand.chromium.png|0| Removed|
|Toolbar.Toolbar Example Variables.chromium.png|0| Removed|
|Toolbar.Toolbar Example.chromium.png|0| Removed|

</details>


  <details><summary><b>Tooltip</b> 46 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Tooltip.TooltipExample.chromium.png|0| Added|
|Tooltip.TooltipExampleDialogContentShorthand.chromium.png|0| Added|
|Tooltip.TooltipExampleDisabledTrigger.chromium.png|0| Added|
|Tooltip.TooltipExampleDismissOnContentMouseEnter.chromium.png|0| Added|
|Tooltip.TooltipExamplePointerMargin.chromium.png|0| Added|
|Tooltip.TooltipExamplePointing.chromium.png|0| Added|
|Tooltip.TooltipExamplePointingTeams.chromium.png|0| Added|
|Tooltip.TooltipExamplePointingTeamsDark.chromium.png|0| Added|
|Tooltip.TooltipExamplePointingTeamsHighContrast.chromium.png|0| Added|
|Tooltip.TooltipExamplePosition.chromium.png|0| Added|
|Tooltip.TooltipExampleRtl.chromium.png|0| Added|
|Tooltip.TooltipExampleTarget.chromium.png|0| Added|
|Tooltip.TooltipExampleTeams.chromium.png|0| Added|
|Tooltip.TooltipExampleTeamsDark.chromium.png|0| Added|
|Tooltip.TooltipExampleTeamsHighContrast.chromium.png|0| Added|
|Tooltip.TooltipOpenExample.chromium.png|0| Added|
|Tooltip.Tooltip Example Dialog Content Shorthand.Shows tooltip in a dialog.chromium.png|0| Removed|
|Tooltip.Tooltip Example Dialog Content Shorthand.chromium.png|0| Removed|
|Tooltip.Tooltip Example Disabled Trigger.chromium.png|0| Removed|
|Tooltip.Tooltip Example Dismiss On Content Mouse Enter.chromium.png|0| Removed|
|Tooltip.Tooltip Example Pointer Margin.chromium.png|0| Removed|
|Tooltip.Tooltip Example Pointing Teams Dark.Has outline on keyboard.chromium.png|0| Removed|
|Tooltip.Tooltip Example Pointing Teams Dark.No outline after click.chromium.png|0| Removed|
|Tooltip.Tooltip Example Pointing Teams Dark.chromium.png|0| Removed|
|Tooltip.Tooltip Example Pointing Teams High Contrast.Has outline on keyboard.chromium.png|0| Removed|
|Tooltip.Tooltip Example Pointing Teams High Contrast.No outline after click.chromium.png|0| Removed|
|Tooltip.Tooltip Example Pointing Teams High Contrast.chromium.png|0| Removed|
|Tooltip.Tooltip Example Pointing Teams.Has outline on keyboard.chromium.png|0| Removed|
|Tooltip.Tooltip Example Pointing Teams.No outline after click.chromium.png|0| Removed|
|Tooltip.Tooltip Example Pointing Teams.chromium.png|0| Removed|
|Tooltip.Tooltip Example Pointing.Has outline on keyboard.chromium.png|0| Removed|
|Tooltip.Tooltip Example Pointing.No outline after click.chromium.png|0| Removed|
|Tooltip.Tooltip Example Pointing.chromium.png|0| Removed|
|Tooltip.Tooltip Example Position.chromium.png|0| Removed|
|Tooltip.Tooltip Example Rtl.RTL- Shows tooltip.chromium.png|0| Removed|
|Tooltip.Tooltip Example Rtl.chromium.png|0| Removed|
|Tooltip.Tooltip Example Target.chromium.png|0| Removed|
|Tooltip.Tooltip Example Teams Dark.Shows tooltip.chromium.png|0| Removed|
|Tooltip.Tooltip Example Teams Dark.chromium.png|0| Removed|
|Tooltip.Tooltip Example Teams High Contrast.Shows tooltip.chromium.png|0| Removed|
|Tooltip.Tooltip Example Teams High Contrast.chromium.png|0| Removed|
|Tooltip.Tooltip Example Teams.Shows tooltip.chromium.png|0| Removed|
|Tooltip.Tooltip Example Teams.chromium.png|0| Removed|
|Tooltip.Tooltip Example.Shows tooltip.chromium.png|0| Removed|
|Tooltip.Tooltip Example.chromium.png|0| Removed|
|Tooltip.Tooltip Open Example.chromium.png|0| Removed|

</details>


  <details><summary><b>Tree</b> 56 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Tree.TreeExampleShorthand.chromium.png|0| Added|
|Tree.TreeExampleShorthandTeams.chromium.png|0| Added|
|Tree.TreeExampleShorthandTeamsDark.chromium.png|0| Added|
|Tree.TreeExampleShorthandTeamsHighContrast.chromium.png|0| Added|
|Tree.TreeExclusiveExample.chromium.png|0| Added|
|Tree.TreeInitiallyOpenExampleShorthand.chromium.png|0| Added|
|Tree.TreeMultiselectExample.chromium.png|0| Added|
|Tree.TreeMultiselectExampleTeams.chromium.png|0| Added|
|Tree.TreeMultiselectExampleTeamsDark.chromium.png|0| Added|
|Tree.TreeMultiselectExampleTeamsHighContrast.chromium.png|0| Added|
|Tree.TreeTitleCustomizationExample.chromium.png|0| Added|
|Tree.TreeTitleCustomizationExampleTeams.chromium.png|0| Added|
|Tree.TreeTitleCustomizationExampleTeamsDark.chromium.png|0| Added|
|Tree.TreeTitleCustomizationExampleTeamsHighContrast.chromium.png|0| Added|
|Tree.Tree Example Shorthand Teams Dark.Focus on click leaf.chromium.png|0| Removed|
|Tree.Tree Example Shorthand Teams Dark.Focus on click subtree.chromium.png|0| Removed|
|Tree.Tree Example Shorthand Teams Dark.Focus on keyboard leaf.chromium.png|0| Removed|
|Tree.Tree Example Shorthand Teams Dark.Focus on keyboard subtree.chromium.png|0| Removed|
|Tree.Tree Example Shorthand Teams Dark.chromium.png|0| Removed|
|Tree.Tree Example Shorthand Teams High Contrast.Focus on click leaf.chromium.png|0| Removed|
|Tree.Tree Example Shorthand Teams High Contrast.Focus on click subtree.chromium.png|0| Removed|
|Tree.Tree Example Shorthand Teams High Contrast.Focus on keyboard leaf.chromium.png|0| Removed|
|Tree.Tree Example Shorthand Teams High Contrast.Focus on keyboard subtree.chromium.png|0| Removed|
|Tree.Tree Example Shorthand Teams High Contrast.chromium.png|0| Removed|
|Tree.Tree Example Shorthand Teams.Focus on click leaf.chromium.png|0| Removed|
|Tree.Tree Example Shorthand Teams.Focus on click subtree.chromium.png|0| Removed|
|Tree.Tree Example Shorthand Teams.Focus on keyboard leaf.chromium.png|0| Removed|
|Tree.Tree Example Shorthand Teams.Focus on keyboard subtree.chromium.png|0| Removed|
|Tree.Tree Example Shorthand Teams.chromium.png|0| Removed|
|Tree.Tree Example Shorthand.Focus on click leaf.chromium.png|0| Removed|
|Tree.Tree Example Shorthand.Focus on click subtree.chromium.png|0| Removed|
|Tree.Tree Example Shorthand.Focus on keyboard leaf.chromium.png|0| Removed|
|Tree.Tree Example Shorthand.Focus on keyboard subtree.chromium.png|0| Removed|
|Tree.Tree Example Shorthand.chromium.png|0| Removed|
|Tree.Tree Exclusive Example.chromium.png|0| Removed|
|Tree.Tree Initially Open Example Shorthand.chromium.png|0| Removed|
|Tree.Tree Multiselect Example Teams Dark.chromium.png|0| Removed|
|Tree.Tree Multiselect Example Teams Dark.default selected states.chromium.png|0| Removed|
|Tree.Tree Multiselect Example Teams Dark.first expanded, not custom checkbox visible.chromium.png|0| Removed|
|Tree.Tree Multiselect Example Teams High Contrast.chromium.png|0| Removed|
|Tree.Tree Multiselect Example Teams High Contrast.default selected states.chromium.png|0| Removed|
|Tree.Tree Multiselect Example Teams High Contrast.first expanded, not custom checkbox visible.chromium.png|0| Removed|
|Tree.Tree Multiselect Example Teams.chromium.png|0| Removed|
|Tree.Tree Multiselect Example Teams.default selected states.chromium.png|0| Removed|
|Tree.Tree Multiselect Example Teams.first expanded, not custom checkbox visible.chromium.png|0| Removed|
|Tree.Tree Multiselect Example.chromium.png|0| Removed|
|Tree.Tree Multiselect Example.default selected states.chromium.png|0| Removed|
|Tree.Tree Multiselect Example.first expanded, not custom checkbox visible.chromium.png|0| Removed|
|Tree.Tree Title Customization Example Teams Dark.Exapanded.chromium.png|0| Removed|
|Tree.Tree Title Customization Example Teams Dark.chromium.png|0| Removed|
|Tree.Tree Title Customization Example Teams High Contrast.Exapanded.chromium.png|0| Removed|
|Tree.Tree Title Customization Example Teams High Contrast.chromium.png|0| Removed|
|Tree.Tree Title Customization Example Teams.Exapanded.chromium.png|0| Removed|
|Tree.Tree Title Customization Example Teams.chromium.png|0| Removed|
|Tree.Tree Title Customization Example.Exapanded.chromium.png|0| Removed|
|Tree.Tree Title Customization Example.chromium.png|0| Removed|

</details>


  <details><summary><b>Video</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Video.VideoExample.chromium.png|0| Added|
|Video.Video Example.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v0"/>